# Genetic Variation In Human Populations

This data set gives genotypes variation of 1066 individuals belonging to
52 predefined populations, for 404 microsatellite markers.

## Usage

``` r
data(hdpg)
```

## Format

`hdpg` is a list of 3 components.  

- tab:

  is a data frame with the genotypes of 1066 individuals encoded with 6
  characters (individuals in row, locus in column), for example ‘123098’
  for a heterozygote carrying alleles ‘123’ and ‘098’, ‘123123’ for a
  homozygote carrying two alleles ‘123’ and, ‘000000’ for a not
  classified locus (missing data).

- ind:

  is a a data frame with 4 columns containing information about the 1066
  individuals: `hdpg$ind$id` containing the Diversity Panel
  identification number of each individual, and three factors
  `hdpg$ind$sex`, `hdpg$ind$population` and `hdpg$ind$region` containing
  the names of the 52 populations belonging to 7 major geographic
  regions (see details).

- locus:

  is a dataframe containing four columns: `hdpg$locus$marknames` a
  vector of names of the microsatellite markers, `hdpg$locus$allbyloc` a
  vector containing the number of alleles by loci,
  `hdpg$locus$chromosome` a factor defining a number for one chromosome
  and, `hdpg$locus$maposition` indicating the position of the locus in
  the chromosome.

## Details

The rows of `hdpg$pop` are the names of the 52 populations belonging to
the geographic regions contained in the rows of `hdpg$region`. The
chosen regions are: America, Asia, Europe, Middle East North Africa,
Oceania, Subsaharan AFRICA.  

The 52 populations are: Adygei, Balochi, Bantu, Basque, Bedouin,
Bergamo, Biaka Pygmies, Brahui, Burusho, Cambodian, Columbian, Dai,
Daur, Druze, French, Han, Hazara, Hezhen, Japanese, Kalash, Karitiana,
Lahu, Makrani, Mandenka, Maya, Mbuti Pygmies, Melanesian, Miaozu,
Mongola, Mozabite, Naxi, NewGuinea, Nilote, Orcadian, Oroqen,
Palestinian, Pathan, Pima, Russian, San, Sardinian, She, Sindhi, Surui,
Tu, Tujia, Tuscan, Uygur, Xibo, Yakut, Yizu, Yoruba.  

`hdpg$freq` is a data frame with 52 rows, corresponding to the 52
populations described above, and 4992 microsatellite markers.

## Source

Extract of data prepared by the Human Diversity Panel Genotypes (invalid
http://research.marshfieldclinic.org/genetics/Freq/FreqInfo.htm)

prepared by Hinda Haned, from data used in: Noah A. Rosenberg, Jonatahan
K. Pritchard, James L. Weber, Howard M. Cabb, Kenneth K. Kidds, Lev A.
Zhivotovsky, Marcus W. Feldman (2002) Genetic Structure of human
Populations *Science*, **298**, 2381–2385.

Lev A. Zhivotovsky, Noah Rosenberg, and Marcus W. Feldman (2003).
Features of Evolution and Expansion of Modern Humans, Inferred from
Genomewide Microsatellite Markers *Am. J. Hum. Genet*, **72**,
1171–1186.

## Examples

``` r
data(hdpg)
names(hdpg)
#> [1] "tab"   "ind"   "locus"
str(hdpg)
#> List of 3
#>  $ tab  :'data.frame':   1066 obs. of  404 variables:
#>   ..$ L001: chr [1:1066] "183174" "183183" "189187" "189176" ...
#>   ..$ L002: chr [1:1066] "156156" "156156" "176176" "156152" ...
#>   ..$ L003: chr [1:1066] "171171" "175175" "171167" "175175" ...
#>   ..$ L004: chr [1:1066] "207195" "207199" "195195" "191183" ...
#>   ..$ L005: chr [1:1066] "133130" "139130" "136133" "133133" ...
#>   ..$ L006: chr [1:1066] "256244" "247247" "247235" "253250" ...
#>   ..$ L007: chr [1:1066] "232228" "216212" "256208" "232224" ...
#>   ..$ L008: chr [1:1066] "269257" "289289" "293269" "289257" ...
#>   ..$ L009: chr [1:1066] "117113" "121117" "117117" "113113" ...
#>   ..$ L010: chr [1:1066] "231227" "235235" "231231" "235231" ...
#>   ..$ L011: chr [1:1066] "166162" "162162" "166158" "174166" ...
#>   ..$ L012: chr [1:1066] "178178" "178174" "178178" "178174" ...
#>   ..$ L013: chr [1:1066] "155147" "147147" "155147" "151147" ...
#>   ..$ L014: chr [1:1066] "126122" "122118" "122118" "122118" ...
#>   ..$ L015: chr [1:1066] "113107" "116104" "116104" "113104" ...
#>   ..$ L016: chr [1:1066] "193190" "193187" "190178" "193187" ...
#>   ..$ L017: chr [1:1066] "208208" "206200" "206200" "210206" ...
#>   ..$ L018: chr [1:1066] "108104" "108100" "116108" "000000" ...
#>   ..$ L019: chr [1:1066] "164156" "152145" "164152" "156152" ...
#>   ..$ L020: chr [1:1066] "200196" "204192" "212204" "200200" ...
#>   ..$ L021: chr [1:1066] "205205" "205205" "208205" "217202" ...
#>   ..$ L022: chr [1:1066] "207203" "219191" "215211" "211203" ...
#>   ..$ L023: chr [1:1066] "242238" "242242" "238238" "238230" ...
#>   ..$ L024: chr [1:1066] "241233" "245241" "237237" "237233" ...
#>   ..$ L025: chr [1:1066] "000000" "189185" "189181" "189181" ...
#>   ..$ L026: chr [1:1066] "260257" "260260" "263260" "260257" ...
#>   ..$ L027: chr [1:1066] "191175" "193189" "189187" "193187" ...
#>   ..$ L028: chr [1:1066] "120120" "120120" "120116" "120120" ...
#>   ..$ L029: chr [1:1066] "192184" "192184" "188184" "192188" ...
#>   ..$ L030: chr [1:1066] "119119" "141141" "137119" "139119" ...
#>   ..$ L031: chr [1:1066] "129117" "129129" "129117" "129117" ...
#>   ..$ L032: chr [1:1066] "316316" "320316" "324312" "320312" ...
#>   ..$ L033: chr [1:1066] "181181" "201181" "181181" "189185" ...
#>   ..$ L034: chr [1:1066] "119119" "119111" "115111" "115111" ...
#>   ..$ L035: chr [1:1066] "164156" "156152" "160156" "148140" ...
#>   ..$ L036: chr [1:1066] "249245" "249245" "249245" "249245" ...
#>   ..$ L037: chr [1:1066] "199195" "195159" "191191" "203195" ...
#>   ..$ L038: chr [1:1066] "249240" "246246" "000000" "246246" ...
#>   ..$ L039: chr [1:1066] "119113" "119113" "119116" "119113" ...
#>   ..$ L040: chr [1:1066] "142139" "142139" "147135" "143142" ...
#>   ..$ L041: chr [1:1066] "170170" "174162" "170170" "174170" ...
#>   ..$ L042: chr [1:1066] "296292" "300296" "304292" "300296" ...
#>   ..$ L043: chr [1:1066] "244228" "236228" "236228" "228228" ...
#>   ..$ L044: chr [1:1066] "168168" "170166" "178168" "174168" ...
#>   ..$ L045: chr [1:1066] "154142" "158158" "158142" "158142" ...
#>   ..$ L046: chr [1:1066] "294290" "290290" "294286" "310290" ...
#>   ..$ L047: chr [1:1066] "153141" "149137" "145141" "141141" ...
#>   ..$ L048: chr [1:1066] "153147" "147144" "156144" "144141" ...
#>   ..$ L049: chr [1:1066] "300288" "292292" "304296" "304296" ...
#>   ..$ L050: chr [1:1066] "137113" "121121" "125125" "125125" ...
#>   ..$ L051: chr [1:1066] "149145" "153141" "157157" "153145" ...
#>   ..$ L052: chr [1:1066] "128124" "132112" "124112" "000000" ...
#>   ..$ L053: chr [1:1066] "278274" "274270" "274270" "278274" ...
#>   ..$ L054: chr [1:1066] "188184" "188172" "172172" "184180" ...
#>   ..$ L055: chr [1:1066] "255251" "255251" "255251" "251247" ...
#>   ..$ L056: chr [1:1066] "183183" "187183" "183183" "183179" ...
#>   ..$ L057: chr [1:1066] "164148" "160154" "164154" "154154" ...
#>   ..$ L058: chr [1:1066] "209197" "185177" "000000" "201197" ...
#>   ..$ L059: chr [1:1066] "178176" "180176" "178176" "180176" ...
#>   ..$ L060: chr [1:1066] "000000" "249243" "251249" "247243" ...
#>   ..$ L061: chr [1:1066] "228208" "228208" "236208" "000000" ...
#>   ..$ L062: chr [1:1066] "260256" "256256" "264256" "284260" ...
#>   ..$ L063: chr [1:1066] "215203" "215199" "215215" "199199" ...
#>   ..$ L064: chr [1:1066] "142142" "146146" "154146" "146146" ...
#>   ..$ L065: chr [1:1066] "194194" "198194" "198198" "198194" ...
#>   ..$ L066: chr [1:1066] "124115" "124115" "115115" "127124" ...
#>   ..$ L067: chr [1:1066] "224220" "228224" "228224" "228224" ...
#>   ..$ L068: chr [1:1066] "000000" "184172" "000000" "172166" ...
#>   ..$ L069: chr [1:1066] "308300" "300292" "300300" "296296" ...
#>   ..$ L070: chr [1:1066] "151151" "159155" "159155" "167163" ...
#>   ..$ L071: chr [1:1066] "188176" "192188" "192176" "200176" ...
#>   ..$ L072: chr [1:1066] "151143" "159159" "163155" "155155" ...
#>   ..$ L073: chr [1:1066] "240234" "243234" "234228" "252234" ...
#>   ..$ L074: chr [1:1066] "237233" "237229" "237229" "237229" ...
#>   ..$ L075: chr [1:1066] "147147" "159143" "159151" "155143" ...
#>   ..$ L076: chr [1:1066] "272256" "256252" "276264" "268256" ...
#>   ..$ L077: chr [1:1066] "272264" "000000" "276272" "276260" ...
#>   ..$ L078: chr [1:1066] "239219" "225225" "239233" "243219" ...
#>   ..$ L079: chr [1:1066] "122112" "116116" "120112" "118118" ...
#>   ..$ L080: chr [1:1066] "278278" "274274" "286266" "282282" ...
#>   ..$ L081: chr [1:1066] "96096" "96096" "96096" "96096" ...
#>   ..$ L082: chr [1:1066] "150144" "146136" "146144" "144144" ...
#>   ..$ L083: chr [1:1066] "186176" "184174" "184174" "184180" ...
#>   ..$ L084: chr [1:1066] "128124" "136124" "124120" "000000" ...
#>   ..$ L085: chr [1:1066] "231227" "225225" "229217" "227227" ...
#>   ..$ L086: chr [1:1066] "178170" "198174" "178170" "174174" ...
#>   ..$ L087: chr [1:1066] "141138" "144138" "141138" "138132" ...
#>   ..$ L088: chr [1:1066] "142138" "178150" "178138" "000000" ...
#>   ..$ L089: chr [1:1066] "189185" "189185" "201189" "189181" ...
#>   ..$ L090: chr [1:1066] "241237" "241241" "245241" "241233" ...
#>   ..$ L091: chr [1:1066] "143135" "131131" "143135" "135135" ...
#>   ..$ L092: chr [1:1066] "174162" "174166" "174162" "166162" ...
#>   ..$ L093: chr [1:1066] "155155" "161155" "158155" "155152" ...
#>   ..$ L094: chr [1:1066] "144140" "132132" "144144" "148140" ...
#>   ..$ L095: chr [1:1066] "225221" "209205" "229221" "217209" ...
#>   ..$ L096: chr [1:1066] "253244" "244244" "244238" "253244" ...
#>   ..$ L097: chr [1:1066] "198198" "190190" "210198" "194190" ...
#>   ..$ L098: chr [1:1066] "202194" "206198" "198194" "202202" ...
#>   ..$ L099: chr [1:1066] "145145" "153145" "149145" "153149" ...
#>   .. [list output truncated]
#>  $ ind  :'data.frame':   1066 obs. of  4 variables:
#>   ..$ id        : Factor w/ 1066 levels "1","3","5","7",..: 1 2 3 4 5 6 7 8 9 10 ...
#>   ..$ sex       : Factor w/ 2 levels "1","2": 1 1 1 1 1 1 1 1 1 1 ...
#>   ..$ population: Factor w/ 52 levels "Adygei","Balochi",..: 8 8 8 8 8 8 8 8 8 8 ...
#>   ..$ region    : Factor w/ 7 levels "America","Asia",..: 2 2 2 2 2 2 2 2 2 2 ...
#>  $ locus:'data.frame':   404 obs. of  4 variables:
#>   ..$ marknames : chr [1:404] "280we5" "ggaa3a07z" "GATA27E01" "GATA29A05" ...
#>   ..$ allbyloc  : int [1:404] 13 12 9 12 9 12 17 14 8 19 ...
#>   ..$ chromosome: Factor w/ 25 levels "1","2","3","4",..: 1 1 1 1 1 1 1 1 1 1 ...
#>   ..$ maposition: num [1:404] 4 16.2 29.9 37.1 46.6 ...
```
