<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><title>Function reference â€¢ ade4</title><!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png"><link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png"><link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png"><link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png"><link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png"><link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png"><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.11/clipboard.min.js" integrity="sha512-7O5pXpc0oCRrxk8RUfDYFgn0nO1t+jLuIOQdOMRp4APB7uZ4vSjspzp5y6YDtDs4VzUSTbWzBFZ/LKJhnyFOKw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Function reference"><meta property="og:image" content="/logo.svg"><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-dark navbar-expand-lg bg-primary" data-bs-theme="dark"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">ade4</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.7-22</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-new-to-ade-">New to ade4?</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-new-to-ade-">
    <a class="dropdown-item" href="../articles/install.html">Installation</a>
    <a class="dropdown-item" href="../articles/get_started.html">Getting started</a>
  </div>
</li>
<li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-vignettes">Vignettes</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-vignettes">
    <a class="external-link dropdown-item" href="https://cran.r-project.org/web/packages/adegraphics/vignettes/adegraphics.html">Graphical representation of multivariate data</a>
    <a class="dropdown-item" href="../articles/ChapEnvVarStruct.html">Environmental variables structures</a>
    <a class="dropdown-item" href="../articles/ChapSpeciesStruct.html">Species structures</a>
    <a class="dropdown-item" href="../articles/ChapBetGr.html">Taking into account groups of sites</a>
    <a class="dropdown-item" href="../articles/Chap2tables.html">Description of species-environment relationships</a>
    <a class="dropdown-item" href="../articles/ChapKtables.html">Analysing Changes in Structures</a>
    <a class="dropdown-item" href="../articles/Chap2Ktables.html">Analysing Changes in Co-structures</a>
  </div>
</li>
<li class="nav-item">
  <a class="nav-link" href="../articles/related.html">Related packages</a>
</li>
<li class="nav-item">
  <a class="nav-link" href="../news/index.html">News</a>
</li>
<li class="active nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-ressources">Ressources</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-ressources">
    <a class="external-link dropdown-item" href="https://lbbe-shiny.univ-lyon1.fr/ade4shiny/">ade4shiny</a>
    <a class="dropdown-item" href="../articles/faq.html">FAQ</a>
    <a class="external-link dropdown-item" href="https://listes.univ-lyon1.fr/sympa/info/adelist">adelist</a>
    <a class="dropdown-item" href="../articles/book.html">ade4 book</a>
    <a class="dropdown-item" href="../articles/papers.html">Papers</a>
  </div>
</li>
      </ul><form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off"></form>

      <ul class="navbar-nav"><li class="nav-item">
  <a class="external-link nav-link" href="https://lbbe-web.univ-lyon1.fr/en">
    <span class="fa fa-leaf"></span>
     
    LBBE
  </a>
</li>
<li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/adeverse/ade4">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul></div>

    
  </div>
</nav><div class="container template-reference-index">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="../logo.svg" class="logo" alt=""><h1>Function reference</h1>
    </div>

    <div class="section level2">
      <h2 id="one-table-analysis">One-table analysis<a class="anchor" aria-label="anchor" href="#one-table-analysis"></a></h2>
      
      <div class="section-desc"><p>These functions are designed for the analysis of a single table.</p></div>

      
    </div><div class="section level2">
      
      <h3 id="methods">Methods<a class="anchor" aria-label="anchor" href="#methods"></a></h3>
      <div class="section-desc"><p>Functions to run the analysis</p></div>

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="dudi.acm.html">dudi.acm()</a></code> <code><a href="dudi.acm.html">acm.burt()</a></code> <code><a href="dudi.acm.html">acm.disjonctif()</a></code> <code><a href="dudi.acm.html">boxplot(<i>&lt;acm&gt;</i>)</a></code> 
        </dt>
        <dd>Multiple Correspondence Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="dudi.coa.html">dudi.coa()</a></code> 
        </dt>
        <dd>Correspondence Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="dudi.dec.html">dudi.dec()</a></code> 
        </dt>
        <dd>Decentred Correspondence Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="dudi.fca.html">prep.fuzzy.var()</a></code> <code><a href="dudi.fca.html">dudi.fca()</a></code> <code><a href="dudi.fca.html">dudi.fpca()</a></code> 
        </dt>
        <dd>Fuzzy Correspondence Analysis and Fuzzy Principal Components Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="dudi.hillsmith.html">dudi.hillsmith()</a></code> 
        </dt>
        <dd>Ordination of Tables mixing quantitative variables and factors</dd>
      </dl><dl><dt>
          
          <code><a href="dudi.mix.html">dudi.mix()</a></code> 
        </dt>
        <dd>Ordination of Tables mixing quantitative variables and factors</dd>
      </dl><dl><dt>
          
          <code><a href="dudi.nsc.html">dudi.nsc()</a></code> 
        </dt>
        <dd>Non symmetric correspondence analysis</dd>
      </dl><dl><dt>
          
          <code><a href="dudi.pca.html">dudi.pca()</a></code> 
        </dt>
        <dd>Principal Component Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="dudi.pco.html">dudi.pco()</a></code> <code><a href="dudi.pco.html">scatter(<i>&lt;pco&gt;</i>)</a></code> 
        </dt>
        <dd>Principal Coordinates Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="pcoscaled.html">pcoscaled()</a></code> 
        </dt>
        <dd>Simplified Analysis in Principal Coordinates</dd>
      </dl><dl><dt>
          
          <code><a href="nipals.html">nipals()</a></code> <code><a href="nipals.html">scatter(<i>&lt;nipals&gt;</i>)</a></code> <code><a href="nipals.html">print(<i>&lt;nipals&gt;</i>)</a></code> 
        </dt>
        <dd>Non-linear Iterative Partial Least Squares (NIPALS) algorithm</dd>
      </dl></div><div class="section level2">
      
      <h3 id="graphics">Graphics<a class="anchor" aria-label="anchor" href="#graphics"></a></h3>
      <div class="section-desc"><p>Functions to display the results of the analysis</p></div>

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="score.coa.html">score(<i>&lt;coa&gt;</i>)</a></code> <code><a href="score.coa.html">reciprocal.coa()</a></code> 
        </dt>
        <dd>Reciprocal scaling after a correspondence analysis</dd>
      </dl><dl><dt>
          
          <code><a href="scatter.html">scatter()</a></code> <code><a href="scatter.html">biplot(<i>&lt;dudi&gt;</i>)</a></code> <code><a href="scatter.html">screeplot(<i>&lt;dudi&gt;</i>)</a></code> 
        </dt>
        <dd>Graphical representation of the outputs of a multivariate analysis</dd>
      </dl><dl><dt>
          
          <code><a href="scatter.acm.html">scatter(<i>&lt;acm&gt;</i>)</a></code> 
        </dt>
        <dd>Plot of the factorial maps in a Multiple Correspondence Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="scatter.coa.html">scatter(<i>&lt;coa&gt;</i>)</a></code> 
        </dt>
        <dd>Plot of the factorial maps for a correspondence analysis</dd>
      </dl><dl><dt>
          
          <code><a href="scatter.dudi.html">scatter(<i>&lt;dudi&gt;</i>)</a></code> 
        </dt>
        <dd>Plot of the Factorial Maps</dd>
      </dl><dl><dt>
          
          <code><a href="scatter.fca.html">scatter(<i>&lt;fca&gt;</i>)</a></code> 
        </dt>
        <dd>Plot of the factorial maps for a fuzzy correspondence analysis</dd>
      </dl><dl><dt>
          
          <code><a href="score.html">score()</a></code> <code><a href="score.html">scoreutil.base()</a></code> 
        </dt>
        <dd>Graphs for One Dimension</dd>
      </dl><dl><dt>
          
          <code><a href="score.acm.html">score(<i>&lt;acm&gt;</i>)</a></code> 
        </dt>
        <dd>Graphs to study one factor in a Multiple Correspondence Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="score.mix.html">score(<i>&lt;mix&gt;</i>)</a></code> 
        </dt>
        <dd>Graphs to Analyse a factor in a Mixed Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="score.pca.html">score(<i>&lt;pca&gt;</i>)</a></code> 
        </dt>
        <dd>Graphs to Analyse a factor in PCA</dd>
      </dl></div><div class="section level2">
      <h2 id="two-table-analysis">Two-table analysis<a class="anchor" aria-label="anchor" href="#two-table-analysis"></a></h2>
      
      <div class="section-desc"><p>These functions are designed for the analysis of a pair of tables.</p></div>

      
    </div><div class="section level2">
      
      <h3 id="methods-1">Methods<a class="anchor" aria-label="anchor" href="#methods-1"></a></h3>
      <div class="section-desc"><p>Functions to run the analysis</p></div>

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="coinertia.html">coinertia()</a></code> <code><a href="coinertia.html">plot(<i>&lt;coinertia&gt;</i>)</a></code> <code><a href="coinertia.html">print(<i>&lt;coinertia&gt;</i>)</a></code> <code><a href="coinertia.html">summary(<i>&lt;coinertia&gt;</i>)</a></code> 
        </dt>
        <dd>Coinertia Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="pcaiv.html">pcaiv()</a></code> <code><a href="pcaiv.html">plot(<i>&lt;pcaiv&gt;</i>)</a></code> <code><a href="pcaiv.html">print(<i>&lt;pcaiv&gt;</i>)</a></code> <code><a href="pcaiv.html">summary(<i>&lt;pcaiv&gt;</i>)</a></code> 
        </dt>
        <dd>Principal component analysis with respect to instrumental variables</dd>
      </dl><dl><dt>
          
          <code><a href="pcaivortho.html">pcaivortho()</a></code> <code><a href="pcaivortho.html">summary(<i>&lt;pcaivortho&gt;</i>)</a></code> 
        </dt>
        <dd>Principal Component Analysis with respect to orthogonal instrumental variables</dd>
      </dl><dl><dt>
          
          <code><a href="procuste.html">procuste()</a></code> <code><a href="procuste.html">plot(<i>&lt;procuste&gt;</i>)</a></code> <code><a href="procuste.html">print(<i>&lt;procuste&gt;</i>)</a></code> <code><a href="procuste.html">randtest(<i>&lt;procuste&gt;</i>)</a></code> 
        </dt>
        <dd>Simple Procruste Rotation between two sets of points</dd>
      </dl><dl><dt>
          
          <code><a href="niche.html">niche()</a></code> <code><a href="niche.html">print(<i>&lt;niche&gt;</i>)</a></code> <code><a href="niche.html">plot(<i>&lt;niche&gt;</i>)</a></code> <code><a href="niche.html">niche.param()</a></code> <code><a href="niche.html">rtest(<i>&lt;niche&gt;</i>)</a></code> 
        </dt>
        <dd>Method to Analyse a pair of tables : Environmental and Faunistic Data</dd>
      </dl><dl><dt>
          
          <code><a href="varipart.html">varipart()</a></code> <code><a href="varipart.html">print(<i>&lt;varipart&gt;</i>)</a></code> 
        </dt>
        <dd>Partition of the variation of a response multivariate table by 2 explanatory tables</dd>
      </dl></div><div class="section level2">
      
      <h3 id="tests">Tests<a class="anchor" aria-label="anchor" href="#tests"></a></h3>
      <div class="section-desc"><p>Functions to evaluate the significance of observed structures</p></div>

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="procuste.randtest.html">procuste.randtest()</a></code> 
        </dt>
        <dd>Monte-Carlo Test on the sum of the singular values of a procustean rotation (in C).</dd>
      </dl><dl><dt>
          
          <code><a href="procuste.rtest.html">procuste.rtest()</a></code> 
        </dt>
        <dd>Monte-Carlo Test on the sum of the singular values of a procustean rotation (in R).</dd>
      </dl><dl><dt>
          
          <code><a href="randtest.coinertia.html">randtest(<i>&lt;coinertia&gt;</i>)</a></code> 
        </dt>
        <dd>Monte-Carlo test on a Co-inertia analysis (in C).</dd>
      </dl><dl><dt>
          
          <code><a href="randtest.pcaiv.html">randtest(<i>&lt;pcaiv&gt;</i>)</a></code> <code><a href="randtest.pcaiv.html">randtest(<i>&lt;pcaivortho&gt;</i>)</a></code> 
        </dt>
        <dd>Monte-Carlo Test on the percentage of explained (i.e. constrained) inertia</dd>
      </dl><dl><dt>
          
          <code><a href="RV.randtest.html">RV.randtest()</a></code> 
        </dt>
        <dd>Monte-Carlo Test on the sum of eigenvalues of a co-inertia analysis (in C++ with Rcpp).</dd>
      </dl><dl><dt>
          
          <code><a href="RV.rtest.html">RV.rtest()</a></code> 
        </dt>
        <dd>Monte-Carlo Test on the sum of eigenvalues of a co-inertia analysis (in R).</dd>
      </dl><dl><dt>
          
          <code><a href="RVintra.randtest.html">RVintra.randtest()</a></code> 
        </dt>
        <dd>Monte-Carlo Test on the sum of eigenvalues of a within-class co-inertia analysis (in C++ with Rcpp).</dd>
      </dl></div><div class="section level2">
      <h2 id="three-table-methods">Three-table methods<a class="anchor" aria-label="anchor" href="#three-table-methods"></a></h2>
      
      <div class="section-desc"><p>3-tables methods.</p></div>

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="combine.4thcorner.html">combine.randtest.rlq()</a></code> <code><a href="combine.4thcorner.html">combine.4thcorner()</a></code> <code><a href="combine.4thcorner.html">p.adjust.4thcorner()</a></code> 
        </dt>
        <dd>Functions to combine and adjust the outputs 3-table methods</dd>
      </dl><dl><dt>
          
          <code><a href="fourthcorner.html">fourthcorner</a></code> <code><a href="fourthcorner.html">fourthcorner2</a></code> <code><a href="fourthcorner.html">print.4thcorner</a></code> <code><a href="fourthcorner.html">summary.4thcorner</a></code> <code><a href="fourthcorner.html">plot.4thcorner</a></code> <code><a href="fourthcorner.html">fourthcorner.rlq</a></code> 
        </dt>
        <dd>Functions to compute the fourth-corner statistic</dd>
      </dl><dl><dt>
          
          <code><a href="rlq.html">rlq()</a></code> <code><a href="rlq.html">print(<i>&lt;rlq&gt;</i>)</a></code> <code><a href="rlq.html">plot(<i>&lt;rlq&gt;</i>)</a></code> <code><a href="rlq.html">summary(<i>&lt;rlq&gt;</i>)</a></code> <code><a href="rlq.html">randtest(<i>&lt;rlq&gt;</i>)</a></code> 
        </dt>
        <dd>RLQ analysis</dd>
      </dl></div><div class="section level2">
      <h2 id="analysis-including-categorical-variables-factor-">Analysis including categorical variables (<code>factor</code>)<a class="anchor" aria-label="anchor" href="#analysis-including-categorical-variables-factor-"></a></h2>
      
      <div class="section-desc"><p>These functions allows to consider a partition of individuals to focus on or remove differences between groups.</p></div>

      
    </div><div class="section level2">
      
      <h3 id="methods-2">Methods<a class="anchor" aria-label="anchor" href="#methods-2"></a></h3>
      

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="bca.html">bca(<i>&lt;dudi&gt;</i>)</a></code> 
        </dt>
        <dd>Between-Class Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="bca.coinertia.html">bca(<i>&lt;coinertia&gt;</i>)</a></code> 
        </dt>
        <dd>Between-class coinertia analysis</dd>
      </dl><dl><dt>
          
          <code><a href="bca.rlq.html">bca(<i>&lt;rlq&gt;</i>)</a></code> <code><a href="bca.rlq.html">plot(<i>&lt;betrlq&gt;</i>)</a></code> <code><a href="bca.rlq.html">print(<i>&lt;betrlq&gt;</i>)</a></code> 
        </dt>
        <dd>Between-Class RLQ analysis</dd>
      </dl><dl><dt>
          
          <code><a href="bwca.dpcoa.html">bwca.dpcoa()</a></code> <code><a href="bwca.dpcoa.html">bca(<i>&lt;dpcoa&gt;</i>)</a></code> <code><a href="bwca.dpcoa.html">wca(<i>&lt;dpcoa&gt;</i>)</a></code> <code><a href="bwca.dpcoa.html">randtest(<i>&lt;betwit&gt;</i>)</a></code> <code><a href="bwca.dpcoa.html">summary(<i>&lt;betwit&gt;</i>)</a></code> <code><a href="bwca.dpcoa.html">print(<i>&lt;witdpcoa&gt;</i>)</a></code> <code><a href="bwca.dpcoa.html">print(<i>&lt;betdpcoa&gt;</i>)</a></code> 
        </dt>
        <dd>Between- and within-class double principal coordinate analysis</dd>
      </dl><dl><dt>
          
          <code><a href="discrimin.html">discrimin()</a></code> <code><a href="discrimin.html">plot(<i>&lt;discrimin&gt;</i>)</a></code> <code><a href="discrimin.html">print(<i>&lt;discrimin&gt;</i>)</a></code> 
        </dt>
        <dd>Linear Discriminant Analysis (descriptive statistic)</dd>
      </dl><dl><dt>
          
          <code><a href="discrimin.coa.html">discrimin.coa()</a></code> 
        </dt>
        <dd>Discriminant Correspondence Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="wca.html">wca(<i>&lt;dudi&gt;</i>)</a></code> 
        </dt>
        <dd>Within-Class Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="wca.coinertia.html">wca(<i>&lt;coinertia&gt;</i>)</a></code> 
        </dt>
        <dd>Within-class coinertia analysis</dd>
      </dl><dl><dt>
          
          <code><a href="wca.rlq.html">wca(<i>&lt;rlq&gt;</i>)</a></code> <code><a href="wca.rlq.html">plot(<i>&lt;witrlq&gt;</i>)</a></code> <code><a href="wca.rlq.html">print(<i>&lt;witrlq&gt;</i>)</a></code> 
        </dt>
        <dd>Within-Class RLQ analysis</dd>
      </dl><dl><dt>
          
          <code><a href="withinpca.html">withinpca()</a></code> 
        </dt>
        <dd>Normed within principal component analysis</dd>
      </dl><dl><dt>
          
          <code><a href="witwit.coa.html">witwit.coa()</a></code> <code><a href="witwit.coa.html">summary(<i>&lt;witwit&gt;</i>)</a></code> <code><a href="witwit.coa.html">witwitsepan()</a></code> 
        </dt>
        <dd>Internal Correspondence Analysis</dd>
      </dl></div><div class="section level2">
      
      <h3 id="additional-tools">Additional tools<a class="anchor" aria-label="anchor" href="#additional-tools"></a></h3>
      

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="loocv.bca.html">loocv(<i>&lt;between&gt;</i>)</a></code> <code><a href="loocv.bca.html">print(<i>&lt;bcaloocv&gt;</i>)</a></code> <code><a href="loocv.bca.html">plot(<i>&lt;bcaloocv&gt;</i>)</a></code> 
        </dt>
        <dd>Leave-one-out cross-validation for a <code>bca</code></dd>
      </dl><dl><dt>
          
          <code><a href="loocv.discrimin.html">loocv(<i>&lt;discrimin&gt;</i>)</a></code> <code><a href="loocv.discrimin.html">print(<i>&lt;discloocv&gt;</i>)</a></code> <code><a href="loocv.discrimin.html">plot(<i>&lt;discloocv&gt;</i>)</a></code> 
        </dt>
        <dd>Leave-one-out cross-validation for a <code>discrimin</code> analysis</dd>
      </dl><dl><dt>
          
          <code><a href="randtest.between.html">randtest(<i>&lt;between&gt;</i>)</a></code> 
        </dt>
        <dd>Monte-Carlo Test on the between-groups inertia percentage (in C).</dd>
      </dl><dl><dt>
          
          <code><a href="randtest.discrimin.html">randtest(<i>&lt;discrimin&gt;</i>)</a></code> 
        </dt>
        <dd>Monte-Carlo Test on a Discriminant Analysis (in C).</dd>
      </dl><dl><dt>
          
          <code><a href="rtest.between.html">rtest(<i>&lt;between&gt;</i>)</a></code> 
        </dt>
        <dd>Monte-Carlo Test on the between-groups inertia percentage (in R).</dd>
      </dl><dl><dt>
          
          <code><a href="rtest.discrimin.html">rtest(<i>&lt;discrimin&gt;</i>)</a></code> 
        </dt>
        <dd>Monte-Carlo Test on a Discriminant Analysis (in R).</dd>
      </dl></div><div class="section level2">
      <h2 id="management-of-dudi-objects">Management of <code>dudi</code> objects<a class="anchor" aria-label="anchor" href="#management-of-dudi-objects"></a></h2>
      
      <div class="section-desc"><p>These functions provides functionalities to manage objects of the class <code>dudi</code> (they can be used on the outputs od several methods to analyze one or two tables for instance).</p></div>

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="dudi.html">as.dudi()</a></code> <code><a href="dudi.html">print(<i>&lt;dudi&gt;</i>)</a></code> <code><a href="dudi.html">is.dudi()</a></code> <code><a href="dudi.html">redo.dudi()</a></code> <code><a href="dudi.html">t(<i>&lt;dudi&gt;</i>)</a></code> <code><a href="dudi.html">summary(<i>&lt;dudi&gt;</i>)</a></code> <code><a href="dudi.html">`[`(<i>&lt;dudi&gt;</i>)</a></code> 
        </dt>
        <dd>Duality Diagram</dd>
      </dl><dl><dt>
          
          <code><a href="scatter.html">scatter()</a></code> <code><a href="scatter.html">biplot(<i>&lt;dudi&gt;</i>)</a></code> <code><a href="scatter.html">screeplot(<i>&lt;dudi&gt;</i>)</a></code> 
        </dt>
        <dd>Graphical representation of the outputs of a multivariate analysis</dd>
      </dl><dl><dt>
          
          <code><a href="dist.dudi.html">dist.dudi()</a></code> 
        </dt>
        <dd>Computation of the Distance Matrix from a Statistical Triplet</dd>
      </dl><dl><dt>
          
          <code><a href="inertia.dudi.html">inertia(<i>&lt;dudi&gt;</i>)</a></code> <code><a href="inertia.dudi.html">print(<i>&lt;inertia&gt;</i>)</a></code> <code><a href="inertia.dudi.html">summary(<i>&lt;inertia&gt;</i>)</a></code> 
        </dt>
        <dd>Decomposition of inertia (i.e. contributions) in multivariate methods</dd>
      </dl><dl><dt>
          
          <code><a href="loocv.dudi.html">loocv(<i>&lt;dudi&gt;</i>)</a></code> 
        </dt>
        <dd>Leave-one-out cross-validation for a <code>dudi</code></dd>
      </dl><dl><dt>
          
          <code><a href="reconst.html">reconst()</a></code> 
        </dt>
        <dd>Reconstitution of Data from a Duality Diagram</dd>
      </dl><dl><dt>
          
          <code><a href="supcol.html">supcol()</a></code> 
        </dt>
        <dd>Projections of Supplementary Columns</dd>
      </dl><dl><dt>
          
          <code><a href="suprow.html">suprow(<i>&lt;coa&gt;</i>)</a></code> <code><a href="suprow.html">suprow(<i>&lt;dudi&gt;</i>)</a></code> <code><a href="suprow.html">predict(<i>&lt;dudi&gt;</i>)</a></code> <code><a href="suprow.html">suprow(<i>&lt;pca&gt;</i>)</a></code> <code><a href="suprow.html">suprow(<i>&lt;acm&gt;</i>)</a></code> <code><a href="suprow.html">suprow(<i>&lt;mix&gt;</i>)</a></code> <code><a href="suprow.html">suprow(<i>&lt;fca&gt;</i>)</a></code> 
        </dt>
        <dd>Projections of Supplementary Rows</dd>
      </dl><dl><dt>
          
          <code><a href="testdim.html">testdim()</a></code> 
        </dt>
        <dd>Function to perform a test of dimensionality</dd>
      </dl></div><div class="section level2">
      <h2 id="k-table-analysis">K-table analysis<a class="anchor" aria-label="anchor" href="#k-table-analysis"></a></h2>
      
      <div class="section-desc"><p>Functions designed to manage and analyze data structured in several tables (a.k.a. multitable analysis)</p></div>

      
    </div><div class="section level2">
      
      <h3 id="methods-3">Methods<a class="anchor" aria-label="anchor" href="#methods-3"></a></h3>
      <div class="section-desc"><p>Functions to run the analysis</p></div>

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="costatis.html">costatis()</a></code> 
        </dt>
        <dd>STATIS and Co-Inertia : Analysis of a series of paired ecological tables</dd>
      </dl><dl><dt>
          
          <code><a href="foucart.html">foucart()</a></code> <code><a href="foucart.html">plot(<i>&lt;foucart&gt;</i>)</a></code> <code><a href="foucart.html">print(<i>&lt;foucart&gt;</i>)</a></code> 
        </dt>
        <dd>K-tables Correspondence Analysis with the same rows and the same columns</dd>
      </dl><dl><dt>
          
          <code><a href="mdpcoa.html">mdpcoa()</a></code> <code><a href="mdpcoa.html">kplotX.mdpcoa()</a></code> <code><a href="mdpcoa.html">prep.mdpcoa()</a></code> 
        </dt>
        <dd>Multiple Double Principal Coordinate Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="mbpls.html">mbpls()</a></code> 
        </dt>
        <dd>Multiblock partial least squares</dd>
      </dl><dl><dt>
          
          <code><a href="mbpcaiv.html">mbpcaiv()</a></code> 
        </dt>
        <dd>Multiblock principal component analysis with instrumental variables</dd>
      </dl><dl><dt>
          
          <code><a href="mcoa.html">mcoa()</a></code> <code><a href="mcoa.html">print(<i>&lt;mcoa&gt;</i>)</a></code> <code><a href="mcoa.html">summary(<i>&lt;mcoa&gt;</i>)</a></code> <code><a href="mcoa.html">plot(<i>&lt;mcoa&gt;</i>)</a></code> 
        </dt>
        <dd>Multiple CO-inertia Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="mfa.html">mfa()</a></code> <code><a href="mfa.html">plot(<i>&lt;mfa&gt;</i>)</a></code> <code><a href="mfa.html">print(<i>&lt;mfa&gt;</i>)</a></code> <code><a href="mfa.html">summary(<i>&lt;mfa&gt;</i>)</a></code> 
        </dt>
        <dd>Multiple Factorial Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="pta.html">pta()</a></code> <code><a href="pta.html">plot(<i>&lt;pta&gt;</i>)</a></code> <code><a href="pta.html">print(<i>&lt;pta&gt;</i>)</a></code> 
        </dt>
        <dd>Partial Triadic Analysis of a K-tables</dd>
      </dl><dl><dt>
          
          <code><a href="sepan.html">sepan()</a></code> <code><a href="sepan.html">plot(<i>&lt;sepan&gt;</i>)</a></code> <code><a href="sepan.html">summary(<i>&lt;sepan&gt;</i>)</a></code> <code><a href="sepan.html">print(<i>&lt;sepan&gt;</i>)</a></code> 
        </dt>
        <dd>Separated Analyses in a K-tables</dd>
      </dl><dl><dt>
          
          <code><a href="statico.html">statico()</a></code> 
        </dt>
        <dd>STATIS and Co-Inertia : Analysis of a series of paired ecological tables</dd>
      </dl><dl><dt>
          
          <code><a href="statis.html">statis()</a></code> <code><a href="statis.html">plot(<i>&lt;statis&gt;</i>)</a></code> <code><a href="statis.html">print(<i>&lt;statis&gt;</i>)</a></code> 
        </dt>
        <dd>STATIS, a method for analysing K-tables</dd>
      </dl></div><div class="section level2">
      
      <h3 id="graphics-1">Graphics<a class="anchor" aria-label="anchor" href="#graphics-1"></a></h3>
      <div class="section-desc"><p>Functions to display the results of the analysis</p></div>

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="kplot.html">kplot()</a></code> 
        </dt>
        <dd>Generic Function for Multiple Graphs in a K-tables Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="kplot.foucart.html">kplot(<i>&lt;foucart&gt;</i>)</a></code> 
        </dt>
        <dd>Multiple Graphs for the Foucart's Correspondence Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="kplot.mcoa.html">kplot(<i>&lt;mcoa&gt;</i>)</a></code> 
        </dt>
        <dd>Multiple Graphs for a Multiple Co-inertia Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="kplot.mfa.html">kplot(<i>&lt;mfa&gt;</i>)</a></code> 
        </dt>
        <dd>Multiple Graphs for a Multiple Factorial Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="kplot.pta.html">kplot(<i>&lt;pta&gt;</i>)</a></code> 
        </dt>
        <dd>Multiple Graphs for a Partial Triadic Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="kplot.sepan.html">kplot(<i>&lt;sepan&gt;</i>)</a></code> <code><a href="kplot.sepan.html">kplotsepan.coa()</a></code> 
        </dt>
        <dd>Multiple Graphs for Separated Analyses in a K-tables</dd>
      </dl><dl><dt>
          
          <code><a href="kplot.statis.html">kplot(<i>&lt;statis&gt;</i>)</a></code> 
        </dt>
        <dd>Multiple Graphs of a STATIS Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="mdpcoa.html">mdpcoa()</a></code> <code><a href="mdpcoa.html">kplotX.mdpcoa()</a></code> <code><a href="mdpcoa.html">prep.mdpcoa()</a></code> 
        </dt>
        <dd>Multiple Double Principal Coordinate Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="multiblock.html">summary(<i>&lt;multiblock&gt;</i>)</a></code> <code><a href="multiblock.html">print(<i>&lt;multiblock&gt;</i>)</a></code> 
        </dt>
        <dd>Display and summarize multiblock objects</dd>
      </dl></div><div class="section level2">
      
      <h3 id="utilities">Utilities<a class="anchor" aria-label="anchor" href="#utilities"></a></h3>
      <div class="section-desc"><p>Functions to manage objects of class <code>ktab</code></p></div>

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="ktab.html">c(<i>&lt;ktab&gt;</i>)</a></code> <code><a href="ktab.html">`[`(<i>&lt;ktab&gt;</i>)</a></code> <code><a href="ktab.html">is.ktab()</a></code> <code><a href="ktab.html">t(<i>&lt;ktab&gt;</i>)</a></code> <code><a href="ktab.html">row.names(<i>&lt;ktab&gt;</i>)</a></code> <code><a href="ktab.html">tab.names()</a></code> <code><a href="ktab.html">col.names()</a></code> <code><a href="ktab.html">ktab.util.names()</a></code> 
        </dt>
        <dd>the class of objects 'ktab' (K-tables)</dd>
      </dl><dl><dt>
          
          <code><a href="ktab.data.frame.html">ktab.data.frame()</a></code> 
        </dt>
        <dd>Creation of K-tables from a data frame</dd>
      </dl><dl><dt>
          
          <code><a href="ktab.list.df.html">ktab.list.df()</a></code> 
        </dt>
        <dd>Creating a K-tables from a list of data frames.</dd>
      </dl><dl><dt>
          
          <code><a href="ktab.list.dudi.html">ktab.list.dudi()</a></code> 
        </dt>
        <dd>Creation of a K-tables from a list of duality diagrams</dd>
      </dl><dl><dt>
          
          <code><a href="ktab.match2ktabs.html">ktab.match2ktabs()</a></code> 
        </dt>
        <dd>STATIS and Co-Inertia : Analysis of a series of paired ecological tables</dd>
      </dl><dl><dt>
          
          <code><a href="ktab.within.html">ktab.within()</a></code> 
        </dt>
        <dd>Process to go from a Within Analysis to a K-tables</dd>
      </dl><dl><dt>
          
          <code><a href="mdpcoa.html">mdpcoa()</a></code> <code><a href="mdpcoa.html">kplotX.mdpcoa()</a></code> <code><a href="mdpcoa.html">prep.mdpcoa()</a></code> 
        </dt>
        <dd>Multiple Double Principal Coordinate Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="suprow.pta.html">suprow(<i>&lt;pta&gt;</i>)</a></code> 
        </dt>
        <dd>Projections of Supplementary Rows for a Partial Triadic Analysis of K-tables</dd>
      </dl></div><div class="section level2">
      
      <h3 id="tests-1">Tests<a class="anchor" aria-label="anchor" href="#tests-1"></a></h3>
      <div class="section-desc"><p>Functions to evaluate the significance of observed structures</p></div>

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="costatis.randtest.html">costatis.randtest()</a></code> 
        </dt>
        <dd>Monte-Carlo test on a Costatis analysis (in C).</dd>
      </dl><dl><dt>
          
          <code><a href="randboot.multiblock.html">randboot(<i>&lt;multiblock&gt;</i>)</a></code> 
        </dt>
        <dd>Bootstraped simulations for multiblock methods</dd>
      </dl><dl><dt>
          
          <code><a href="testdim.multiblock.html">testdim(<i>&lt;multiblock&gt;</i>)</a></code> 
        </dt>
        <dd>Selection of the number of dimension by two-fold cross-validation for multiblock methods</dd>
      </dl><dl><dt>
          
          <code><a href="statico.krandtest.html">statico.krandtest()</a></code> 
        </dt>
        <dd>Monte-Carlo test on a Statico analysis (in C).</dd>
      </dl></div><div class="section level2">
      <h2 id="distance-methods">Distance methods<a class="anchor" aria-label="anchor" href="#distance-methods"></a></h2>
      
      

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="cailliez.html">cailliez()</a></code> 
        </dt>
        <dd>Transformation to make Euclidean a distance matrix</dd>
      </dl><dl><dt>
          
          <code><a href="dist.binary.html">dist.binary()</a></code> 
        </dt>
        <dd>Computation of Distance Matrices for Binary Data</dd>
      </dl><dl><dt>
          
          <code><a href="dist.dudi.html">dist.dudi()</a></code> 
        </dt>
        <dd>Computation of the Distance Matrix from a Statistical Triplet</dd>
      </dl><dl><dt>
          
          <code><a href="dist.ktab.html">dist.ktab()</a></code> <code><a href="dist.ktab.html">ldist.ktab()</a></code> <code><a href="dist.ktab.html">kdist.cor()</a></code> <code><a href="dist.ktab.html">prep.fuzzy()</a></code> <code><a href="dist.ktab.html">prep.binary()</a></code> <code><a href="dist.ktab.html">prep.circular()</a></code> 
        </dt>
        <dd>Mixed-variables coefficient of distance</dd>
      </dl><dl><dt>
          
          <code><a href="dist.neig.html">dist.neig()</a></code> 
        </dt>
        <dd>Computation of the Distance Matrix associated to a Neighbouring Graph</dd>
      </dl><dl><dt>
          
          <code><a href="dist.prop.html">dist.prop()</a></code> 
        </dt>
        <dd>Computation of Distance Matrices of Percentage Data</dd>
      </dl><dl><dt>
          
          <code><a href="dist.quant.html">dist.quant()</a></code> 
        </dt>
        <dd>Computation of Distance Matrices on Quantitative Variables</dd>
      </dl><dl><dt>
          
          <code><a href="as.taxo.html">as.taxo()</a></code> <code><a href="as.taxo.html">dist.taxo()</a></code> 
        </dt>
        <dd>Taxonomy</dd>
      </dl><dl><dt>
          
          <code><a href="dudi.pco.html">dudi.pco()</a></code> <code><a href="dudi.pco.html">scatter(<i>&lt;pco&gt;</i>)</a></code> 
        </dt>
        <dd>Principal Coordinates Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="kdist.html">kdist()</a></code> 
        </dt>
        <dd>the class of objects 'kdist' (K distance matrices)</dd>
      </dl><dl><dt>
          
          <code><a href="kdist2ktab.html">kdist2ktab()</a></code> 
        </dt>
        <dd>Transformation of K distance matrices (object 'kdist') into K Euclidean representations (object 'ktab')</dd>
      </dl><dl><dt>
          
          <code><a href="kdisteuclid.html">kdisteuclid()</a></code> 
        </dt>
        <dd>a way to obtain Euclidean distance matrices</dd>
      </dl><dl><dt>
          
          <code><a href="mantel.randtest.html">mantel.randtest()</a></code> 
        </dt>
        <dd>Mantel test (correlation between two distance matrices (in C).)</dd>
      </dl><dl><dt>
          
          <code><a href="mantel.rtest.html">mantel.rtest()</a></code> 
        </dt>
        <dd>Mantel test (correlation between two distance matrices (in R).)</dd>
      </dl><dl><dt>
          
          <code><a href="corkdist.html">mantelkdist()</a></code> <code><a href="corkdist.html">RVkdist()</a></code> <code><a href="corkdist.html">plot(<i>&lt;corkdist&gt;</i>)</a></code> 
        </dt>
        <dd>Tests of randomization between distances applied to 'kdist' objetcs</dd>
      </dl><dl><dt>
          
          <code><a href="quasieuclid.html">quasieuclid()</a></code> 
        </dt>
        <dd>Transformation of a distance matrice to a Euclidean one</dd>
      </dl><dl><dt>
          
          <code><a href="RVdist.randtest.html">RVdist.randtest()</a></code> 
        </dt>
        <dd>Tests of randomization on the correlation between two distance matrices (in R).</dd>
      </dl><dl><dt>
          
          <code><a href="supdist.html">supdist()</a></code> 
        </dt>
        <dd>Projection of additional items in a PCO analysis</dd>
      </dl><dl><dt>
          
          <code><a href="table.dist.html">table.dist()</a></code> 
        </dt>
        <dd>Graph Display for Distance Matrices</dd>
      </dl><dl><dt>
          
          <code><a href="is.euclid.html">is.euclid()</a></code> <code><a href="is.euclid.html">summary(<i>&lt;dist&gt;</i>)</a></code> 
        </dt>
        <dd>Is a Distance Matrix Euclidean?</dd>
      </dl><dl><dt>
          
          <code><a href="lingoes.html">lingoes()</a></code> 
        </dt>
        <dd>Transformation of a Distance Matrix for becoming Euclidean</dd>
      </dl><dl><dt>
          
          <code><a href="mstree.html">mstree()</a></code> 
        </dt>
        <dd>Minimal Spanning Tree</dd>
      </dl><dl><dt>
          
          <code><a href="pcoscaled.html">pcoscaled()</a></code> 
        </dt>
        <dd>Simplified Analysis in Principal Coordinates</dd>
      </dl></div><div class="section level2">
      <h2 id="graphics-2">Graphics<a class="anchor" aria-label="anchor" href="#graphics-2"></a></h2>
      
      <div class="section-desc"><p>Functions to draw graphics. See functions of the package <code>adegraphics</code> for a more flexible implementation based on <code>lattice</code>.</p></div>

      
    </div><div class="section level2">
      
      <h3 id="-d-plot">1D plot<a class="anchor" aria-label="anchor" href="#-d-plot"></a></h3>
      

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="sco.boxplot.html">sco.boxplot()</a></code> 
        </dt>
        <dd>Representation of the link between a variable and a set of qualitative variables</dd>
      </dl><dl><dt>
          
          <code><a href="sco.class.html">sco.class()</a></code> 
        </dt>
        <dd>1D plot of a numeric score and a factor with labels</dd>
      </dl><dl><dt>
          
          <code><a href="sco.distri.html">sco.distri()</a></code> 
        </dt>
        <dd>Representation by mean- standard deviation of a set of weight distributions on a numeric score</dd>
      </dl><dl><dt>
          
          <code><a href="sco.gauss.html">sco.gauss()</a></code> 
        </dt>
        <dd>Relationships between one score and qualitative variables</dd>
      </dl><dl><dt>
          
          <code><a href="sco.quant.html">sco.quant()</a></code> 
        </dt>
        <dd>Graph to Analyse the Relation between a Score and Quantitative Variables</dd>
      </dl><dl><dt>
          
          <code><a href="sco.label.html">sco.label()</a></code> 
        </dt>
        <dd>1D plot of a numeric score with labels</dd>
      </dl><dl><dt>
          
          <code><a href="sco.match.html">sco.match()</a></code> 
        </dt>
        <dd>1D plot of a pair of numeric scores with labels</dd>
      </dl></div><div class="section level2">
      
      <h3 id="-d-plot-1">2D plot<a class="anchor" aria-label="anchor" href="#-d-plot-1"></a></h3>
      

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="s.arrow.html">s.arrow()</a></code> 
        </dt>
        <dd>Plot of the factorial maps for the projection of a vector basis</dd>
      </dl><dl><dt>
          
          <code><a href="s.chull.html">s.chull()</a></code> 
        </dt>
        <dd>Plot of the factorial maps with polygons of contour by level of a factor</dd>
      </dl><dl><dt>
          
          <code><a href="s.class.html">s.class()</a></code> 
        </dt>
        <dd>Plot of factorial maps with representation of point classes</dd>
      </dl><dl><dt>
          
          <code><a href="s.corcircle.html">s.corcircle()</a></code> 
        </dt>
        <dd>Plot of the factorial maps of a correlation circle</dd>
      </dl><dl><dt>
          
          <code><a href="s.distri.html">s.distri()</a></code> 
        </dt>
        <dd>Plot of a frequency distribution</dd>
      </dl><dl><dt>
          
          <code><a href="s.hist.html">s.hist()</a></code> 
        </dt>
        <dd>Display of a scatterplot and its two marginal histograms</dd>
      </dl><dl><dt>
          
          <code><a href="s.image.html">s.image()</a></code> 
        </dt>
        <dd>Graph of a variable using image and contour</dd>
      </dl><dl><dt>
          
          <code><a href="s.kde2d.html">s.kde2d()</a></code> 
        </dt>
        <dd>Scatter Plot with Kernel Density Estimate</dd>
      </dl><dl><dt>
          
          <code><a href="s.label.html">s.label()</a></code> 
        </dt>
        <dd>Scatter Plot</dd>
      </dl><dl><dt>
          
          <code><a href="s.logo.html">s.logo()</a></code> 
        </dt>
        <dd>Representation of an object in a graph by a picture</dd>
      </dl><dl><dt>
          
          <code><a href="s.match.html">s.match()</a></code> 
        </dt>
        <dd>Plot of Paired Coordinates</dd>
      </dl><dl><dt>
          
          <code><a href="s.match.class.html">s.match.class()</a></code> 
        </dt>
        <dd>Scatterplot of two sets of coordinates and a partionning into classes</dd>
      </dl><dl><dt>
          
          <code><a href="s.multinom.html">s.multinom()</a></code> 
        </dt>
        <dd>Graph of frequency profiles (useful for instance in genetic)</dd>
      </dl><dl><dt>
          
          <code><a href="s.traject.html">s.traject()</a></code> 
        </dt>
        <dd>Trajectory Plot</dd>
      </dl><dl><dt>
          
          <code><a href="s.value.html">s.value()</a></code> 
        </dt>
        <dd>Representation of a value in a graph</dd>
      </dl></div><div class="section level2">
      
      <h3 id="other-types-of-plot">Other types of plot<a class="anchor" aria-label="anchor" href="#other-types-of-plot"></a></h3>
      

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="dotcircle.html">dotcircle()</a></code> 
        </dt>
        <dd>Representation of n values on a circle</dd>
      </dl><dl><dt>
          
          <code><a href="table.cont.html">table.cont()</a></code> 
        </dt>
        <dd>Plot of Contingency Tables</dd>
      </dl><dl><dt>
          
          <code><a href="table.dist.html">table.dist()</a></code> 
        </dt>
        <dd>Graph Display for Distance Matrices</dd>
      </dl><dl><dt>
          
          <code><a href="table.paint.html">table.paint()</a></code> 
        </dt>
        <dd>Plot of the arrays by grey levels</dd>
      </dl><dl><dt>
          
          <code><a href="table.value.html">table.value()</a></code> 
        </dt>
        <dd>Plot of the Arrays</dd>
      </dl><dl><dt>
          
          <code><a href="triangle.plot.html">triangle.plot()</a></code> <code><a href="triangle.plot.html">triangle.biplot()</a></code> 
        </dt>
        <dd>Triangular Plotting</dd>
      </dl><dl><dt>
          
          <code><a href="triangle.class.html">triangle.class()</a></code> 
        </dt>
        <dd>Triangular Representation and Groups of points</dd>
      </dl></div><div class="section level2">
      
      <h3 id="utilities-1">Utilities<a class="anchor" aria-label="anchor" href="#utilities-1"></a></h3>
      <div class="section-desc"><p>These functions are mainly designed for internal use.</p></div>

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="add.scatter.html">add.scatter()</a></code> <code><a href="add.scatter.html">add.scatter.eig()</a></code> 
        </dt>
        <dd>Add graphics to an existing plot</dd>
      </dl><dl><dt>
          
          <code><a href="scatterutil.html">scatterutil</a></code> <code><a href="scatterutil.html">scatterutil.base</a></code> <code><a href="scatterutil.html">scatterutil.sco</a></code> <code><a href="scatterutil.html">scatterutil.chull</a></code> <code><a href="scatterutil.html">scatterutil.eigen</a></code> <code><a href="scatterutil.html">scatterutil.ellipse</a></code> <code><a href="scatterutil.html">scatterutil.eti.circ</a></code> <code><a href="scatterutil.html">scatterutil.eti</a></code> <code><a href="scatterutil.html">scatterutil.grid</a></code> <code><a href="scatterutil.html">scatterutil.legend.bw.square</a></code> <code><a href="scatterutil.html">scatterutil.legend.square.grey</a></code> <code><a href="scatterutil.html">scatterutil.legendgris</a></code> <code><a href="scatterutil.html">scatterutil.scaling</a></code> <code><a href="scatterutil.html">scatterutil.star</a></code> <code><a href="scatterutil.html">scatterutil.sub</a></code> <code><a href="scatterutil.html">scatterutil.convrot90</a></code> 
        </dt>
        <dd>Graphical utility functions</dd>
      </dl><dl><dt>
          
          <code><a href="s.logo.html">s.logo()</a></code> 
        </dt>
        <dd>Representation of an object in a graph by a picture</dd>
      </dl></div><div class="section level2">
      
      <h3 id="diversity">Diversity<a class="anchor" aria-label="anchor" href="#diversity"></a></h3>
      <div class="section-desc"><p>Functions focusing on the measurment of diversity. See also the package <code>adiv</code> for more functionalities.</p></div>

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="amova.html">amova()</a></code> <code><a href="amova.html">print(<i>&lt;amova&gt;</i>)</a></code> 
        </dt>
        <dd>Analysis of molecular variance</dd>
      </dl><dl><dt>
          
          <code><a href="apqe.html">apqe()</a></code> <code><a href="apqe.html">print(<i>&lt;apqe&gt;</i>)</a></code> 
        </dt>
        <dd>Apportionment of Quadratic Entropy</dd>
      </dl><dl><dt>
          
          <code><a href="disc.html">disc()</a></code> 
        </dt>
        <dd>Rao's dissimilarity coefficient</dd>
      </dl><dl><dt>
          
          <code><a href="divc.html">divc()</a></code> 
        </dt>
        <dd>Rao's diversity coefficient also called quadratic entropy</dd>
      </dl><dl><dt>
          
          <code><a href="divcmax.html">divcmax()</a></code> 
        </dt>
        <dd>Maximal value of Rao's diversity coefficient also called quadratic entropy</dd>
      </dl><dl><dt>
          
          <code><a href="dpcoa.html">dpcoa()</a></code> <code><a href="dpcoa.html">plot(<i>&lt;dpcoa&gt;</i>)</a></code> <code><a href="dpcoa.html">print(<i>&lt;dpcoa&gt;</i>)</a></code> <code><a href="dpcoa.html">summary(<i>&lt;dpcoa&gt;</i>)</a></code> 
        </dt>
        <dd>Double principal coordinate analysis</dd>
      </dl><dl><dt>
          
          <code><a href="originality.html">originality()</a></code> 
        </dt>
        <dd>Originality of a species</dd>
      </dl><dl><dt>
          
          <code><a href="randtest.dpcoa.html">randtest(<i>&lt;dpcoa&gt;</i>)</a></code> 
        </dt>
        <dd>Permutation test for double principal coordinate analysis (DPCoA)</dd>
      </dl><dl><dt>
          
          <code><a href="randtest.amova.html">randtest(<i>&lt;amova&gt;</i>)</a></code> 
        </dt>
        <dd>Permutation tests on an analysis of molecular variance (in C).</dd>
      </dl></div><div class="section level2">
      
      <h3 id="spatial">Spatial<a class="anchor" aria-label="anchor" href="#spatial"></a></h3>
      <div class="section-desc"><p>Functions focusing on the analysis of spatial data. See also the package <code>adespatial</code> for more functionalities.</p></div>

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="gridrowcol.html">gridrowcol()</a></code> 
        </dt>
        <dd>Complete regular grid analysis</dd>
      </dl><dl><dt>
          
          <code><a href="mld.html">mld()</a></code> <code><a href="mld.html">haar2level()</a></code> 
        </dt>
        <dd>Multi Level Decomposition of unidimensional data</dd>
      </dl><dl><dt>
          
          <code><a href="mstree.html">mstree()</a></code> 
        </dt>
        <dd>Minimal Spanning Tree</dd>
      </dl><dl><dt>
          
          <code><a href="orthobasis.html">orthobasis.neig()</a></code> <code><a href="orthobasis.html">orthobasis.line()</a></code> <code><a href="orthobasis.html">orthobasis.circ()</a></code> <code><a href="orthobasis.html">orthobasis.mat()</a></code> <code><a href="orthobasis.html">orthobasis.haar()</a></code> <code><a href="orthobasis.html">print(<i>&lt;orthobasis&gt;</i>)</a></code> <code><a href="orthobasis.html">plot(<i>&lt;orthobasis&gt;</i>)</a></code> <code><a href="orthobasis.html">summary(<i>&lt;orthobasis&gt;</i>)</a></code> <code><a href="orthobasis.html">is.orthobasis()</a></code> 
        </dt>
        <dd>Orthonormal basis for orthonormal transform</dd>
      </dl><dl><dt>
          
          <code><a href="neig.html">neig()</a></code> <code><a href="neig.html">scores.neig()</a></code> <code><a href="neig.html">print(<i>&lt;neig&gt;</i>)</a></code> <code><a href="neig.html">summary(<i>&lt;neig&gt;</i>)</a></code> <code><a href="neig.html">nb2neig()</a></code> <code><a href="neig.html">neig2nb()</a></code> <code><a href="neig.html">neig2mat()</a></code> 
        </dt>
        <dd>Neighbourhood Graphs</dd>
      </dl></div><div class="section level2">
      <h2 id="phylogenetic-methods">Phylogenetic methods<a class="anchor" aria-label="anchor" href="#phylogenetic-methods"></a></h2>
      
      <div class="section-desc"><p>These functions allows to manage phylogenetic data. See functions of the package <code>adephylo</code> for a more flexible implementation based on <code>ape</code> and <code>phylobase</code>.</p></div>

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="dotchart.phylog.html">dotchart.phylog()</a></code> 
        </dt>
        <dd>Representation of many quantitative variables in front of a phylogenetic tree</dd>
      </dl><dl><dt>
          
          <code><a href="plot.phylog.html">plot(<i>&lt;phylog&gt;</i>)</a></code> <code><a href="plot.phylog.html">radial.phylog()</a></code> <code><a href="plot.phylog.html">enum.phylog()</a></code> 
        </dt>
        <dd>Plot phylogenies</dd>
      </dl><dl><dt>
          
          <code><a href="gearymoran.html">gearymoran()</a></code> 
        </dt>
        <dd>Moran's I and Geary'c randomization tests for spatial and phylogenetic autocorrelation</dd>
      </dl><dl><dt>
          
          <code><a href="newick2phylog.html">newick2phylog()</a></code> <code><a href="newick2phylog.html">hclust2phylog()</a></code> <code><a href="newick2phylog.html">taxo2phylog()</a></code> <code><a href="newick2phylog.html">newick2phylog.addtools()</a></code> 
        </dt>
        <dd>Create phylogeny</dd>
      </dl><dl><dt>
          
          <code><a href="phylog.html">print(<i>&lt;phylog&gt;</i>)</a></code> <code><a href="phylog.html">phylog.extract()</a></code> <code><a href="phylog.html">phylog.permut()</a></code> 
        </dt>
        <dd>Phylogeny</dd>
      </dl><dl><dt>
          
          <code><a href="symbols.phylog.html">symbols.phylog()</a></code> 
        </dt>
        <dd>Representation of a quantitative variable in front of a phylogenetic tree</dd>
      </dl><dl><dt>
          
          <code><a href="table.phylog.html">table.phylog()</a></code> 
        </dt>
        <dd>Plot arrays in front of a phylogenetic tree</dd>
      </dl><dl><dt>
          
          <code><a href="variance.phylog.html">variance.phylog()</a></code> 
        </dt>
        <dd>The phylogenetic ANOVA</dd>
      </dl><dl><dt>
          
          <code><a href="PI2newick.html">PI2newick()</a></code> 
        </dt>
        <dd>Import data files from Phylogenetic Independance Package</dd>
      </dl></div><div class="section level2">
      <h2 id="tests-2">Tests<a class="anchor" aria-label="anchor" href="#tests-2"></a></h2>
      
      <div class="section-desc"><p>Utilities functions to manage results of randomization procedures.</p></div>

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="randboot.html">as.krandboot()</a></code> <code><a href="randboot.html">print(<i>&lt;krandboot&gt;</i>)</a></code> <code><a href="randboot.html">as.randboot()</a></code> <code><a href="randboot.html">print(<i>&lt;randboot&gt;</i>)</a></code> <code><a href="randboot.html">randboot()</a></code> 
        </dt>
        <dd>Bootstrap simulations</dd>
      </dl><dl><dt>
          
          <code><a href="krandtest.html">as.krandtest()</a></code> <code><a href="krandtest.html">plot(<i>&lt;krandtest&gt;</i>)</a></code> <code><a href="krandtest.html">print(<i>&lt;krandtest&gt;</i>)</a></code> <code><a href="krandtest.html">`[`(<i>&lt;krandtest&gt;</i>)</a></code> <code><a href="krandtest.html">`[[`(<i>&lt;krandtest&gt;</i>)</a></code> 
        </dt>
        <dd>Class of the Permutation Tests (in C).</dd>
      </dl><dl><dt>
          
          <code><a href="randxval.html">as.krandxval()</a></code> <code><a href="randxval.html">print(<i>&lt;krandxval&gt;</i>)</a></code> <code><a href="randxval.html">as.randxval()</a></code> <code><a href="randxval.html">print(<i>&lt;randxval&gt;</i>)</a></code> 
        </dt>
        <dd>Two-fold cross-validation</dd>
      </dl><dl><dt>
          
          <code><a href="randtest.html">randtest()</a></code> <code><a href="randtest.html">as.randtest()</a></code> <code><a href="randtest.html">plot(<i>&lt;randtest&gt;</i>)</a></code> <code><a href="randtest.html">print(<i>&lt;randtest&gt;</i>)</a></code> 
        </dt>
        <dd>Class of the Permutation Tests (in C).</dd>
      </dl><dl><dt>
          
          <code><a href="rtest.html">rtest()</a></code> 
        </dt>
        <dd>Class of the Permutation Tests (in R).</dd>
      </dl></div><div class="section level2">
      <h2 id="misc">Misc<a class="anchor" aria-label="anchor" href="#misc"></a></h2>
      
      

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="ade4.package.html">ade4-package</a></code> <code><a href="ade4.package.html">ade4</a></code> 
        </dt>
        <dd>The ade4 package</dd>
      </dl><dl><dt>
          
          <code><a href="dudi.acm.html">dudi.acm()</a></code> <code><a href="dudi.acm.html">acm.burt()</a></code> <code><a href="dudi.acm.html">acm.disjonctif()</a></code> <code><a href="dudi.acm.html">boxplot(<i>&lt;acm&gt;</i>)</a></code> 
        </dt>
        <dd>Multiple Correspondence Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="adegraphicsLoaded.html">adegraphicsLoaded()</a></code> 
        </dt>
        <dd>Utility function to test if the adegraphics package is loaded</dd>
      </dl><dl><dt>
          
          <code><a href="bicenter.wt.html">bicenter.wt()</a></code> 
        </dt>
        <dd>Double Weighted Centring</dd>
      </dl><dl><dt>
          
          <code><a href="scalewt.html">covwt()</a></code> <code><a href="scalewt.html">varwt()</a></code> <code><a href="scalewt.html">scalewt()</a></code> <code><a href="scalewt.html">meanfacwt()</a></code> <code><a href="scalewt.html">varfacwt()</a></code> <code><a href="scalewt.html">covfacwt()</a></code> <code><a href="scalewt.html">scalefacwt()</a></code> 
        </dt>
        <dd>Compute or scale data using (weighted) means, variances and covariances (possibly for the levels of a factor)</dd>
      </dl><dl><dt>
          
          <code><a href="dagnelie.test.html">dagnelie.test()</a></code> 
        </dt>
        <dd>Dagnelie multinormality test</dd>
      </dl><dl><dt>
          
          <code><a href="uniquewt.df.html">uniquewt.df()</a></code> 
        </dt>
        <dd>Elimination of Duplicated Rows in a Array</dd>
      </dl></div><div class="section level2">
      <h2 id="datasets">Datasets<a class="anchor" aria-label="anchor" href="#datasets"></a></h2>
      
      <div class="section-desc"><p>Datasets available in ade4. You can access them by typing <code>r data(dataset)</code>.</p></div>

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="abouheif.eg.html">abouheif.eg</a></code> 
        </dt>
        <dd>Phylogenies and quantitative traits from Abouheif</dd>
      </dl><dl><dt>
          
          <code><a href="acacia.html">acacia</a></code> 
        </dt>
        <dd>Spatial pattern analysis in plant communities</dd>
      </dl><dl><dt>
          
          <code><a href="aminoacyl.html">aminoacyl</a></code> 
        </dt>
        <dd>Codon usage</dd>
      </dl><dl><dt>
          
          <code><a href="apis108.html">apis108</a></code> 
        </dt>
        <dd>Allelic frequencies in ten honeybees populations at eight microsatellites loci</dd>
      </dl><dl><dt>
          
          <code><a href="aravo.html">aravo</a></code> 
        </dt>
        <dd>Distribution of Alpine plants in Aravo (Valloire, France)</dd>
      </dl><dl><dt>
          
          <code><a href="ardeche.html">ardeche</a></code> 
        </dt>
        <dd>Fauna Table with double (row and column) partitioning</dd>
      </dl><dl><dt>
          
          <code><a href="arrival.html">arrival</a></code> 
        </dt>
        <dd>Arrivals at an intensive care unit</dd>
      </dl><dl><dt>
          
          <code><a href="atlas.html">atlas</a></code> 
        </dt>
        <dd>Small Ecological Dataset</dd>
      </dl><dl><dt>
          
          <code><a href="atya.html">atya</a></code> 
        </dt>
        <dd>Genetic variability of Cacadors</dd>
      </dl><dl><dt>
          
          <code><a href="avijons.html">avijons</a></code> 
        </dt>
        <dd>Bird species distribution</dd>
      </dl><dl><dt>
          
          <code><a href="avimedi.html">avimedi</a></code> 
        </dt>
        <dd>Fauna Table for Constrained Ordinations</dd>
      </dl><dl><dt>
          
          <code><a href="aviurba.html">aviurba</a></code> 
        </dt>
        <dd>Ecological Tables Triplet</dd>
      </dl><dl><dt>
          
          <code><a href="bacteria.html">bacteria</a></code> 
        </dt>
        <dd>Genomes of 43 Bacteria</dd>
      </dl><dl><dt>
          
          <code><a href="banque.html">banque</a></code> 
        </dt>
        <dd>Table of Factors</dd>
      </dl><dl><dt>
          
          <code><a href="baran95.html">baran95</a></code> 
        </dt>
        <dd>African Estuary Fishes</dd>
      </dl><dl><dt>
          
          <code><a href="bf88.html">bf88</a></code> 
        </dt>
        <dd>Cubic Ecological Data</dd>
      </dl><dl><dt>
          
          <code><a href="bordeaux.html">bordeaux</a></code> 
        </dt>
        <dd>Wine Tasting</dd>
      </dl><dl><dt>
          
          <code><a href="bsetal97.html">bsetal97</a></code> 
        </dt>
        <dd>Ecological and Biological Traits</dd>
      </dl><dl><dt>
          
          <code><a href="buech.html">buech</a></code> 
        </dt>
        <dd>Buech basin</dd>
      </dl><dl><dt>
          
          <code><a href="butterfly.html">butterfly</a></code> 
        </dt>
        <dd>Genetics-Ecology-Environment Triple</dd>
      </dl><dl><dt>
          
          <code><a href="capitales.html">capitales</a></code> 
        </dt>
        <dd>Road Distances</dd>
      </dl><dl><dt>
          
          <code><a href="carni19.html">carni19</a></code> 
        </dt>
        <dd>Phylogeny and quantative trait of carnivora</dd>
      </dl><dl><dt>
          
          <code><a href="carni70.html">carni70</a></code> 
        </dt>
        <dd>Phylogeny and quantitative traits of carnivora</dd>
      </dl><dl><dt>
          
          <code><a href="carniherbi49.html">carniherbi49</a></code> 
        </dt>
        <dd>Taxonomy, phylogenies and quantitative traits of carnivora and herbivora</dd>
      </dl><dl><dt>
          
          <code><a href="casitas.html">casitas</a></code> 
        </dt>
        <dd>Enzymatic polymorphism in Mus musculus</dd>
      </dl><dl><dt>
          
          <code><a href="chatcat.html">chatcat</a></code> 
        </dt>
        <dd>Qualitative Weighted Variables</dd>
      </dl><dl><dt>
          
          <code><a href="chats.html">chats</a></code> 
        </dt>
        <dd>Pair of Variables</dd>
      </dl><dl><dt>
          
          <code><a href="chazeb.html">chazeb</a></code> 
        </dt>
        <dd>Charolais-Zebus</dd>
      </dl><dl><dt>
          
          <code><a href="chevaine.html">chevaine</a></code> 
        </dt>
        <dd>Enzymatic polymorphism in Leuciscus cephalus</dd>
      </dl><dl><dt>
          
          <code><a href="chickenk.html">chickenk</a></code> 
        </dt>
        <dd>Veterinary epidemiological study to assess the risk factors for losses in broiler chickens</dd>
      </dl><dl><dt>
          
          <code><a href="clementines.html">clementines</a></code> 
        </dt>
        <dd>Fruit Production</dd>
      </dl><dl><dt>
          
          <code><a href="cnc2003.html">cnc2003</a></code> 
        </dt>
        <dd>Frequenting movie theaters in France in 2003</dd>
      </dl><dl><dt>
          
          <code><a href="coleo.html">coleo</a></code> 
        </dt>
        <dd>Table of Fuzzy Biological Traits</dd>
      </dl><dl><dt>
          
          <code><a href="corvus.html">corvus</a></code> 
        </dt>
        <dd>Corvus morphology</dd>
      </dl><dl><dt>
          
          <code><a href="deug.html">deug</a></code> 
        </dt>
        <dd>Exam marks for some students</dd>
      </dl><dl><dt>
          
          <code><a href="doubs.html">doubs</a></code> 
        </dt>
        <dd>Pair of Ecological Tables</dd>
      </dl><dl><dt>
          
          <code><a href="dunedata.html">dunedata</a></code> 
        </dt>
        <dd>Dune Meadow Data</dd>
      </dl><dl><dt>
          
          <code><a href="ecg.html">ecg</a></code> 
        </dt>
        <dd>Electrocardiogram data</dd>
      </dl><dl><dt>
          
          <code><a href="ecomor.html">ecomor</a></code> 
        </dt>
        <dd>Ecomorphological Convergence</dd>
      </dl><dl><dt>
          
          <code><a href="elec88.html">elec88</a></code> 
        </dt>
        <dd>Electoral Data</dd>
      </dl><dl><dt>
          
          <code><a href="escopage.html">escopage</a></code> 
        </dt>
        <dd>K-tables of wine-tasting</dd>
      </dl><dl><dt>
          
          <code><a href="euro123.html">euro123</a></code> 
        </dt>
        <dd>Triangular Data</dd>
      </dl><dl><dt>
          
          <code><a href="fission.html">fission</a></code> 
        </dt>
        <dd>Fission pattern and heritable morphological traits</dd>
      </dl><dl><dt>
          
          <code><a href="friday87.html">friday87</a></code> 
        </dt>
        <dd>Faunistic K-tables</dd>
      </dl><dl><dt>
          
          <code><a href="fruits.html">fruits</a></code> 
        </dt>
        <dd>Pair of Tables</dd>
      </dl><dl><dt>
          
          <code><a href="ggtortoises.html">ggtortoises</a></code> 
        </dt>
        <dd>Microsatellites of Galapagos tortoises populations</dd>
      </dl><dl><dt>
          
          <code><a href="granulo.html">granulo</a></code> 
        </dt>
        <dd>Granulometric Curves</dd>
      </dl><dl><dt>
          
          <code><a href="hdpg.html">hdpg</a></code> 
        </dt>
        <dd>Genetic Variation In Human Populations</dd>
      </dl><dl><dt>
          
          <code><a href="houmousr.html">houmousr</a></code> 
        </dt>
        <dd>Morphometric data set</dd>
      </dl><dl><dt>
          
          <code><a href="housetasks.html">housetasks</a></code> 
        </dt>
        <dd>Contingency Table</dd>
      </dl><dl><dt>
          
          <code><a href="humDNAm.html">humDNAm</a></code> 
        </dt>
        <dd>human mitochondrial DNA restriction data</dd>
      </dl><dl><dt>
          
          <code><a href="ichtyo.html">ichtyo</a></code> 
        </dt>
        <dd>Point sampling of fish community</dd>
      </dl><dl><dt>
          
          <code><a href="irishdata.html">irishdata</a></code> 
        </dt>
        <dd>Geary's Irish Data</dd>
      </dl><dl><dt>
          
          <code><a href="julliot.html">julliot</a></code> 
        </dt>
        <dd>Seed dispersal</dd>
      </dl><dl><dt>
          
          <code><a href="jv73.html">jv73</a></code> 
        </dt>
        <dd>K-tables Multi-Regions</dd>
      </dl><dl><dt>
          
          <code><a href="kcponds.html">kcponds</a></code> 
        </dt>
        <dd>Ponds in a nature reserve</dd>
      </dl><dl><dt>
          
          <code><a href="lascaux.html">lascaux</a></code> 
        </dt>
        <dd>Genetic/Environment and types of variables</dd>
      </dl><dl><dt>
          
          <code><a href="lizards.html">lizards</a></code> 
        </dt>
        <dd>Phylogeny and quantitative traits of lizards</dd>
      </dl><dl><dt>
          
          <code><a href="macaca.html">macaca</a></code> 
        </dt>
        <dd>Landmarks</dd>
      </dl><dl><dt>
          
          <code><a href="macon.html">macon</a></code> 
        </dt>
        <dd>Wine Tasting</dd>
      </dl><dl><dt>
          
          <code><a href="macroloire.html">macroloire</a></code> 
        </dt>
        <dd>Assemblages of Macroinvertebrates in the Loire River (France)</dd>
      </dl><dl><dt>
          
          <code><a href="mafragh.html">mafragh</a></code> 
        </dt>
        <dd>Phyto-Ecological Survey</dd>
      </dl><dl><dt>
          
          <code><a href="maples.html">maples</a></code> 
        </dt>
        <dd>Phylogeny and quantitative traits of flowers</dd>
      </dl><dl><dt>
          
          <code><a href="mariages.html">mariages</a></code> 
        </dt>
        <dd>Correspondence Analysis Table</dd>
      </dl><dl><dt>
          
          <code><a href="meau.html">meau</a></code> 
        </dt>
        <dd>Ecological Data : sites-variables, sites-species, where and when</dd>
      </dl><dl><dt>
          
          <code><a href="meaudret.html">meaudret</a></code> 
        </dt>
        <dd>Ecological Data : sites-variables, sites-species, where and when</dd>
      </dl><dl><dt>
          
          <code><a href="microsatt.html">microsatt</a></code> 
        </dt>
        <dd>Genetic Relationships between cattle breeds with microsatellites</dd>
      </dl><dl><dt>
          
          <code><a href="mjrochet.html">mjrochet</a></code> 
        </dt>
        <dd>Phylogeny and quantitative traits of teleos fishes</dd>
      </dl><dl><dt>
          
          <code><a href="mollusc.html">mollusc</a></code> 
        </dt>
        <dd>Faunistic Communities and Sampling Experiment</dd>
      </dl><dl><dt>
          
          <code><a href="monde84.html">monde84</a></code> 
        </dt>
        <dd>Global State of the World in 1984</dd>
      </dl><dl><dt>
          
          <code><a href="morphosport.html">morphosport</a></code> 
        </dt>
        <dd>Athletes' Morphology</dd>
      </dl><dl><dt>
          
          <code><a href="newick.eg.html">newick.eg</a></code> 
        </dt>
        <dd>Phylogenetic trees in Newick format</dd>
      </dl><dl><dt>
          
          <code><a href="njplot.html">njplot</a></code> 
        </dt>
        <dd>Phylogeny and trait of bacteria</dd>
      </dl><dl><dt>
          
          <code><a href="olympic.html">olympic</a></code> 
        </dt>
        <dd>Olympic Decathlon</dd>
      </dl><dl><dt>
          
          <code><a href="oribatid.html">oribatid</a></code> 
        </dt>
        <dd>Oribatid mite</dd>
      </dl><dl><dt>
          
          <code><a href="ours.html">ours</a></code> 
        </dt>
        <dd>A table of Qualitative Variables</dd>
      </dl><dl><dt>
          
          <code><a href="palm.html">palm</a></code> 
        </dt>
        <dd>Phylogenetic and quantitative traits of amazonian palm trees</dd>
      </dl><dl><dt>
          
          <code><a href="pap.html">pap</a></code> 
        </dt>
        <dd>Taxonomy and quantitative traits of carnivora</dd>
      </dl><dl><dt>
          
          <code><a href="pcw.html">pcw</a></code> 
        </dt>
        <dd>Distribution of of tropical trees along the Panama canal</dd>
      </dl><dl><dt>
          
          <code><a href="perthi02.html">perthi02</a></code> 
        </dt>
        <dd>Contingency Table with a partition in Molecular Biology</dd>
      </dl><dl><dt>
          
          <code><a href="piosphere.html">piosphere</a></code> 
        </dt>
        <dd>Plant traits response to grazing</dd>
      </dl><dl><dt>
          
          <code><a href="presid2002.html">presid2002</a></code> 
        </dt>
        <dd>Results of the French presidential elections of 2002</dd>
      </dl><dl><dt>
          
          <code><a href="procella.html">procella</a></code> 
        </dt>
        <dd>Phylogeny and quantitative traits of birds</dd>
      </dl><dl><dt>
          
          <code><a href="rankrock.html">rankrock</a></code> 
        </dt>
        <dd>Ordination Table</dd>
      </dl><dl><dt>
          
          <code><a href="rhizobium.html">rhizobium</a></code> 
        </dt>
        <dd>Genetic structure of two nitrogen fixing bacteria influenced by geographical isolation and host specialization</dd>
      </dl><dl><dt>
          
          <code><a href="rhone.html">rhone</a></code> 
        </dt>
        <dd>Physico-Chemistry Data</dd>
      </dl><dl><dt>
          
          <code><a href="rpjdl.html">rpjdl</a></code> 
        </dt>
        <dd>Avifauna and Vegetation</dd>
      </dl><dl><dt>
          
          <code><a href="santacatalina.html">santacatalina</a></code> 
        </dt>
        <dd>Indirect Ordination</dd>
      </dl><dl><dt>
          
          <code><a href="sarcelles.html">sarcelles</a></code> 
        </dt>
        <dd>Array of Recapture of Rings</dd>
      </dl><dl><dt>
          
          <code><a href="seconde.html">seconde</a></code> 
        </dt>
        <dd>Students and Subjects</dd>
      </dl><dl><dt>
          
          <code><a href="skulls.html">skulls</a></code> 
        </dt>
        <dd>Morphometric Evolution</dd>
      </dl><dl><dt>
          
          <code><a href="steppe.html">steppe</a></code> 
        </dt>
        <dd>Transect in the Vegetation</dd>
      </dl><dl><dt>
          
          <code><a href="syndicats.html">syndicats</a></code> 
        </dt>
        <dd>Two Questions asked on a Sample of 1000 Respondents</dd>
      </dl><dl><dt>
          
          <code><a href="t3012.html">t3012</a></code> 
        </dt>
        <dd>Average temperatures of 30 French cities</dd>
      </dl><dl><dt>
          
          <code><a href="tarentaise.html">tarentaise</a></code> 
        </dt>
        <dd>Mountain Avifauna</dd>
      </dl><dl><dt>
          
          <code><a href="taxo.eg.html">taxo.eg</a></code> 
        </dt>
        <dd>Examples of taxonomy</dd>
      </dl><dl><dt>
          
          <code><a href="tintoodiel.html">tintoodiel</a></code> 
        </dt>
        <dd>Tinto and Odiel estuary geochemistry</dd>
      </dl><dl><dt>
          
          <code><a href="tithonia.html">tithonia</a></code> 
        </dt>
        <dd>Phylogeny and quantitative traits of flowers</dd>
      </dl><dl><dt>
          
          <code><a href="tortues.html">tortues</a></code> 
        </dt>
        <dd>Morphological Study of the Painted Turtle</dd>
      </dl><dl><dt>
          
          <code><a href="toxicity.html">toxicity</a></code> 
        </dt>
        <dd>Homogeneous Table</dd>
      </dl><dl><dt>
          
          <code><a href="trichometeo.html">trichometeo</a></code> 
        </dt>
        <dd>Pair of Ecological Data</dd>
      </dl><dl><dt>
          
          <code><a href="ungulates.html">ungulates</a></code> 
        </dt>
        <dd>Phylogeny and quantitative traits of ungulates.</dd>
      </dl><dl><dt>
          
          <code><a href="vegtf.html">vegtf</a></code> 
        </dt>
        <dd>Vegetation in Trois-Fontaines</dd>
      </dl><dl><dt>
          
          <code><a href="veuvage.html">veuvage</a></code> 
        </dt>
        <dd>Example for Centring in PCA</dd>
      </dl><dl><dt>
          
          <code><a href="westafrica.html">westafrica</a></code> 
        </dt>
        <dd>Freshwater fish zoogeography in west Africa</dd>
      </dl><dl><dt>
          
          <code><a href="woangers.html">woangers</a></code> 
        </dt>
        <dd>Plant assemblages in woodlands of the conurbation of Angers (France)</dd>
      </dl><dl><dt>
          
          <code><a href="worksurv.html">worksurv</a></code> 
        </dt>
        <dd>French Worker Survey (1970)</dd>
      </dl><dl><dt>
          
          <code><a href="yanomama.html">yanomama</a></code> 
        </dt>
        <dd>Distance Matrices</dd>
      </dl><dl><dt>
          
          <code><a href="zealand.html">zealand</a></code> 
        </dt>
        <dd>Road distances in New-Zealand</dd>
      </dl></div><div class="section level2">
      <h2 id="deprecated">Deprecated<a class="anchor" aria-label="anchor" href="#deprecated"></a></h2>
      
      

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="ade4-deprecated.html">ade4-deprecated</a></code> 
        </dt>
        <dd>Deprecated functions in ade4</dd>
      </dl><dl><dt>
          
          <code><a href="plot.between.html">plot(<i>&lt;between&gt;</i>)</a></code> <code><a href="plot.between.html">print(<i>&lt;between&gt;</i>)</a></code> <code><a href="plot.between.html">plot(<i>&lt;betcoi&gt;</i>)</a></code> <code><a href="plot.between.html">print(<i>&lt;betcoi&gt;</i>)</a></code> <code><a href="plot.between.html">summary(<i>&lt;between&gt;</i>)</a></code> 
        </dt>
        <dd>Between-Class Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="plot.within.html">plot(<i>&lt;within&gt;</i>)</a></code> <code><a href="plot.within.html">print(<i>&lt;within&gt;</i>)</a></code> <code><a href="plot.within.html">plot(<i>&lt;witcoi&gt;</i>)</a></code> <code><a href="plot.within.html">print(<i>&lt;witcoi&gt;</i>)</a></code> <code><a href="plot.within.html">summary(<i>&lt;within&gt;</i>)</a></code> 
        </dt>
        <dd>Within-Class Analysis</dd>
      </dl></div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside></div>


    <footer><div class="pkgdown-footer-left">
  <p>Developed by StÃ©phane Dray, Anne-BÃ©atrice Dufour, Jean Thioulouse, AurÃ©lie Siberchicot.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.9.</p>
</div>

    </footer></div>

  

  

  </body></html>

