<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><meta name="description" content="The data set concerns fixing bacteria belonging to the genus Sinorhizobium
(Rhizobiaceae) associated with the plant genus Medicago
(Fabaceae). It is a combination of two data sets fully available online from GenBank and
published in two recent papers (see reference below). The complete sampling
procedure is described in the Additional file 3 of the reference below. We delineated six populations
according to geographical origin (France: F, Tunisia Hadjeb:
TH, Tunisia Enfidha: TE), the host plant (M. truncatula
or similar symbiotic specificity: T, M. laciniata: L), and the
taxonomical status of bacteria (S. meliloti: mlt, S. medicae:
mdc). Each population will be called hereafter according
to the three above criteria, e.g. THLmlt is the population
sampled in Tunisia at Hadjeb from M. laciniata nodules
which include S. meliloti isolates. S. medicae interacts with
M. truncatula while S. meliloti interacts with both M. laciniata
(S. meliloti bv. medicaginis) and M. truncatula (S.
meliloti bv. meliloti). The numbers of individuals
are respectively 46 for FTmdc, 43 for FTmlt, 20 for TETmdc,
24 for TETmlt, 20 for TELmlt, 42 for THTmlt and 20
for THLmlt.
Four different intergenic spacers (IGS), IGSNOD,
IGSEXO, IGSGAB, and IGSRKP, distributed on the different
replication units of the model strain 1021 of S.
meliloti bv. meliloti had been sequenced to
characterize each bacterial isolate (DNA extraction and
sequencing procedures are described in an additional file). 
It is noteworthy that the IGSNOD
marker is located within the nod gene cluster and that specific
alleles at these loci determine the ability of S. meliloti
strains to interact with either M. laciniata or M. truncatula."><title>Genetic structure of two nitrogen fixing bacteria influenced
by geographical isolation and host specialization — rhizobium • ade4</title><!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png"><link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png"><link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png"><link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png"><link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png"><link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png"><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.2.2/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.2.2/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Genetic structure of two nitrogen fixing bacteria influenced
by geographical isolation and host specialization — rhizobium"><meta property="og:description" content="The data set concerns fixing bacteria belonging to the genus Sinorhizobium
(Rhizobiaceae) associated with the plant genus Medicago
(Fabaceae). It is a combination of two data sets fully available online from GenBank and
published in two recent papers (see reference below). The complete sampling
procedure is described in the Additional file 3 of the reference below. We delineated six populations
according to geographical origin (France: F, Tunisia Hadjeb:
TH, Tunisia Enfidha: TE), the host plant (M. truncatula
or similar symbiotic specificity: T, M. laciniata: L), and the
taxonomical status of bacteria (S. meliloti: mlt, S. medicae:
mdc). Each population will be called hereafter according
to the three above criteria, e.g. THLmlt is the population
sampled in Tunisia at Hadjeb from M. laciniata nodules
which include S. meliloti isolates. S. medicae interacts with
M. truncatula while S. meliloti interacts with both M. laciniata
(S. meliloti bv. medicaginis) and M. truncatula (S.
meliloti bv. meliloti). The numbers of individuals
are respectively 46 for FTmdc, 43 for FTmlt, 20 for TETmdc,
24 for TETmlt, 20 for TELmlt, 42 for THTmlt and 20
for THLmlt.
Four different intergenic spacers (IGS), IGSNOD,
IGSEXO, IGSGAB, and IGSRKP, distributed on the different
replication units of the model strain 1021 of S.
meliloti bv. meliloti had been sequenced to
characterize each bacterial isolate (DNA extraction and
sequencing procedures are described in an additional file). 
It is noteworthy that the IGSNOD
marker is located within the nod gene cluster and that specific
alleles at these loci determine the ability of S. meliloti
strains to interact with either M. laciniata or M. truncatula."><meta property="og:image" content="/logo.svg"><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-dark navbar-expand-lg bg-primary"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">ade4</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">1.7-22</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-new-to-ade-">New to ade4?</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-new-to-ade-">
    <a class="dropdown-item" href="../articles/install.html">Installation</a>
    <a class="dropdown-item" href="../articles/get_started.html">Getting started</a>
  </div>
</li>
<li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-vignettes">Vignettes</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-vignettes">
    <a class="external-link dropdown-item" href="https://cran.r-project.org/web/packages/adegraphics/vignettes/adegraphics.html">Graphical representation of multivariate data</a>
    <a class="dropdown-item" href="../articles/ChapEnvVarStruct.html">Environmental variables structures</a>
    <a class="dropdown-item" href="../articles/ChapSpeciesStruct.html">Species structures</a>
    <a class="dropdown-item" href="../articles/ChapBetGr.html">Taking into account groups of sites</a>
    <a class="dropdown-item" href="../articles/Chap2tables.html">Description of species-environment relationships</a>
    <a class="dropdown-item" href="../articles/ChapKtables.html">Analysing Changes in Structures</a>
  </div>
</li>
<li class="nav-item">
  <a class="nav-link" href="../articles/related.html">Related packages</a>
</li>
<li class="nav-item">
  <a class="nav-link" href="../news/index.html">News</a>
</li>
<li class="active nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-ressources">Ressources</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-ressources">
    <a class="external-link dropdown-item" href="https://lbbe-shiny.univ-lyon1.fr/Reproducible_Research/ShinyADE4/">ade4 shinyapps</a>
    <a class="dropdown-item" href="../articles/faq.html">FAQ</a>
    <a class="external-link dropdown-item" href="https://listes.univ-lyon1.fr/sympa/info/adelist">adelist</a>
    <a class="dropdown-item" href="../articles/book.html">ade4 book</a>
    <a class="dropdown-item" href="../articles/papers.html">Papers</a>
  </div>
</li>
      </ul><form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off"></form>

      <ul class="navbar-nav"><li class="nav-item">
  <a class="external-link nav-link" href="https://lbbe-web.univ-lyon1.fr/en">
    <span class="fa fa-leaf"></span>
     
    LBBE
  </a>
</li>
<li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/sdray/ade4">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul></div>

    
  </div>
</nav><div class="container template-reference-topic">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="../logo.svg" class="logo" alt=""><h1>Genetic structure of two nitrogen fixing bacteria influenced
by geographical isolation and host specialization</h1>
      
      <div class="d-none name"><code>rhizobium.Rd</code></div>
    </div>

    <div class="ref-description section level2">
    <p>The data set concerns fixing bacteria belonging to the genus Sinorhizobium
(Rhizobiaceae) associated with the plant genus Medicago
(Fabaceae). It is a combination of two data sets fully available online from GenBank and
published in two recent papers (see reference below). The complete sampling
procedure is described in the Additional file 3 of the reference below. We delineated six populations
according to geographical origin (France: F, Tunisia Hadjeb:
TH, Tunisia Enfidha: TE), the host plant (<em>M. truncatula</em>
or similar symbiotic specificity: T, M. laciniata: L), and the
taxonomical status of bacteria (S. meliloti: mlt, S. medicae:
mdc). Each population will be called hereafter according
to the three above criteria, e.g. THLmlt is the population
sampled in Tunisia at Hadjeb from M. laciniata nodules
which include S. meliloti isolates. S. medicae interacts with
M. truncatula while S. meliloti interacts with both M. laciniata
(S. meliloti bv. medicaginis) and M. truncatula (S.
meliloti bv. meliloti). The numbers of individuals
are respectively 46 for FTmdc, 43 for FTmlt, 20 for TETmdc,
24 for TETmlt, 20 for TELmlt, 42 for THTmlt and 20
for THLmlt.</p>
<p>Four different intergenic spacers (IGS), IGSNOD,
IGSEXO, IGSGAB, and IGSRKP, distributed on the different
replication units of the model strain 1021 of S.
meliloti bv. meliloti had been sequenced to
characterize each bacterial isolate (DNA extraction and
sequencing procedures are described in an additional file). 
It is noteworthy that the IGSNOD
marker is located within the nod gene cluster and that specific
alleles at these loci determine the ability of S. meliloti
strains to interact with either M. laciniata or M. truncatula.</p>
    </div>

    <div class="section level2">
    <h2 id="ref-usage">Usage<a class="anchor" aria-label="anchor" href="#ref-usage"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">rhizobium</span><span class="op">)</span></span></code></pre></div>
    </div>

    <div class="section level2">
    <h2 id="format">Format<a class="anchor" aria-label="anchor" href="#format"></a></h2>
    <p><code>rhizobium</code> is a list of 2 components.</p><ul><li><p>dnaobj: list of dna lists. Each dna list corresponds to a locus. For a given locus, the dna list provides the
    dna sequences The ith sequences of all loci corresponds to the ith individual of the data set.</p></li>
<li><p>pop: The list of the populations which each individual sequence belongs to.</p></li>
</ul></div>
    <div class="section level2">
    <h2 id="source">Source<a class="anchor" aria-label="anchor" href="#source"></a></h2>
    <p>Pavoine, S. and Bailly, X. (2007) New analysis for consistency among markers in the study of genetic diversity: 
  development and application to the description of bacterial diversity. <em>BMC Evolutionary Biology</em>, <b>7</b>, e156.</p>
    </div>

    <div class="section level2">
    <h2 id="ref-examples">Examples<a class="anchor" aria-label="anchor" href="#ref-examples"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co"># The functions used below require the package ape</span></span></span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/utils/data.html" class="external-link">data</a></span><span class="op">(</span><span class="va">rhizobium</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="kw">if</span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/ns-load.html" class="external-link">requireNamespace</a></span><span class="op">(</span><span class="st">"ape"</span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span> <span class="op">{</span></span></span>
<span class="r-in"><span><span class="va">dat</span> <span class="op">&lt;-</span> <span class="fu"><a href="mdpcoa.html">prep.mdpcoa</a></span><span class="op">(</span><span class="va">rhizobium</span><span class="op">[[</span><span class="fl">1</span><span class="op">]</span><span class="op">]</span>, <span class="va">rhizobium</span><span class="op">[[</span><span class="fl">2</span><span class="op">]</span><span class="op">]</span>, </span></span>
<span class="r-in"><span>    model <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"F84"</span>, <span class="st">"F84"</span>, <span class="st">"F84"</span>, <span class="st">"F81"</span><span class="op">)</span>,</span></span>
<span class="r-in"><span>    pairwise.deletion <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="va">sam</span> <span class="op">&lt;-</span> <span class="va">dat</span><span class="op">$</span><span class="va">sam</span></span></span>
<span class="r-in"><span><span class="va">dis</span> <span class="op">&lt;-</span> <span class="va">dat</span><span class="op">$</span><span class="va">dis</span></span></span>
<span class="r-in"><span><span class="co"># The distances should be Euclidean. </span></span></span>
<span class="r-in"><span><span class="co"># Several transformations exist to render a distance object Euclidean </span></span></span>
<span class="r-in"><span><span class="co"># (see functions cailliez, lingoes and quasieuclid in the ade4 package). </span></span></span>
<span class="r-in"><span><span class="co"># Here we use the quasieuclid function.</span></span></span>
<span class="r-in"><span><span class="va">dis</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/lapply.html" class="external-link">lapply</a></span><span class="op">(</span><span class="va">dis</span>, <span class="va">quasieuclid</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="va">mdpcoa1</span> <span class="op">&lt;-</span> <span class="fu"><a href="mdpcoa.html">mdpcoa</a></span><span class="op">(</span><span class="va">sam</span>, <span class="va">dis</span>, scann <span class="op">=</span> <span class="cn">FALSE</span>, nf <span class="op">=</span> <span class="fl">2</span><span class="op">)</span></span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co"># Reference analysis</span></span></span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="va">mdpcoa1</span><span class="op">)</span></span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co"># Differences between the loci</span></span></span>
<span class="r-in"><span><span class="fu"><a href="kplot.html">kplot</a></span><span class="op">(</span><span class="va">mdpcoa1</span><span class="op">)</span></span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co"># Alleles projected on the population maps.</span></span></span>
<span class="r-in"><span><span class="fu"><a href="mdpcoa.html">kplotX.mdpcoa</a></span><span class="op">(</span><span class="va">mdpcoa1</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="op">}</span></span></span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>Zero distance(s)</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>Zero distance(s)</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>Zero distance(s)</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>Zero distance(s)</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>Zero distance(s)</span>
<span class="r-err co"><span class="r-pr">#&gt;</span> <span class="error">Error in s.match(dfxy1 = mdpcoa1$Tl1, dfxy2 = as.data.frame(matrix(kronecker(c(1, 1, 1, 1), as.matrix(mdpcoa1$SynVar)), nrow = 28L, ncol = 2L,     dimnames = list(c("FTmdc.EXO.aa", "FTmlt.EXO.aa", "TELmlt.EXO.aa",     "TETmdc.EXO.aa", "TETmlt.EXO.aa", "THLmlt.EXO.aa", "THTmlt.EXO.aa",     "FTmdc.GAB.aa", "FTmlt.GAB.aa", "TELmlt.GAB.aa", "TETmdc.GAB.aa",     "TETmlt.GAB.aa", "THLmlt.GAB.aa", "THTmlt.GAB.aa", "FTmdc.NOD.aa",     "FTmlt.NOD.aa", "TELmlt.NOD.aa", "TETmdc.NOD.aa", "TETmlt.NOD.aa",     "THLmlt.NOD.aa", "THTmlt.NOD.aa", "FTmdc.RKP.aa", "FTmlt.RKP.aa",     "TELmlt.RKP.aa", "TETmdc.RKP.aa", "TETmlt.RKP.aa", "THLmlt.RKP.aa",     "THTmlt.RKP.aa"), c("Axis1", "Axis2")))), xax = 1, yax = 2,     plot = FALSE, storeData = TRUE, pos = -3, psub = list(text = "Rows"),     parrows = list(angle = 0), plabels = list(alpha = 0, boxes = list(        draw = FALSE))):</span> non convenient selection for dfxy1 or dfxy2 (can not be converted to dataframe)</span>
<span class="r-in"><span></span></span>
</code></pre></div>
    </div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside></div>


    <footer><div class="pkgdown-footer-left">
  <p></p><p>Developed by Stéphane Dray, Anne-Béatrice Dufour, Jean Thioulouse, Aurélie Siberchicot.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

    </footer></div>

  

  

  </body></html>

